<!DOCTYPE html>
<html>

<head>

<meta charset="utf-8">

<title>short.annotated</title>


<script type="text/javascript" language="javascript" src='https://code.jquery.com/jquery-3.5.1.js'></script>

<script type="text/javascript" language="javascript" src='https://cdn.datatables.net/1.10.22/js/jquery.dataTables.min.js'></script>

<script type="text/javascript" language="javascript" src='https://cdn.datatables.net/fixedcolumns/3.3.2/js/dataTables.fixedColumns.min.js'></script>

<script type="text/javascript" > 


var openTab;

//document ready
$(document).ready(function() {
				var height = $(window).height();
				var h = Math.floor(height - 250);
				var filterHash = new Object();
				var keyType = "3";
				var keyAnnotID = "0";
				var fullMode = 'full';	
				var dblClickMode = 'off';	
				var oldStart = 0;

				//input FILTER
				var head = '#tabFULLSPLIT thead';
				$(head+' tr').clone(true).appendTo(head);
	
				$(head+' tr:eq(1) th').each( function (i) {
					$(this).removeClass('sorting').removeClass('sorting_asc').removeClass('sorting_desc');
					$(this).unbind();
					let title = $(this).text();
					$(this).html( '<input id="col'+i+'" type="text" placeholder="Search" data-index="'+i+'" />' );
				}); //END FILTER



				//INITIALISATION
				var tabFULLSPLIT = $('#tabFULLSPLIT').DataTable({
					stateSave: true,
					stateDuration: 0,
					order: [],
					paging: true,
					orderCellsTop: true,
					scrollX: true,
					scrollY: h,
					scrollCollapse: true,
					oLanguage: { sLengthMenu: 'Show _MENU_ lines',sInfo: 'Showing _START_ to _END_ of _TOTAL_ lines' },
					lengthMenu: [
							[50, 100, -1],
							[50, 100, 'All']
						],
					fixedColumns:   {
						leftColumns: 1,
						heightMatch: 'auto',
					 },
					stateLoadParams: function( settings, data ) {
						filterHash = data.filter;
						fullMode = data.fullMode ;	
						dblClickMode = data.dblClickMode ;	
						//console.log('load__');
					},
					stateSaveParams: function( settings, data ) {
						data.filter = filterHash;	
						data.fullMode = fullMode;	
						data.dblClickMode = dblClickMode;	
						//console.log('save__');
					},
					drawCallback: function (o) {
						var newStart = this.api().page.info().start;
						if ( newStart != oldStart ) {
							$('.dataTables_scrollBody').scrollTop(0);
							oldStart = newStart;
						}
					},
			});  //END initialisation

			//FILTER FUNCTION
			$(tabFULLSPLIT.table().container() ).on( 'keyup change', 'thead input', function (){ 

						//console.log('toto1');
						var expr = this.value;
						var i = $(this).data('index');
						if(/^[!<>=]+$/.test(expr)){
							return;
						}
						
						if(expr === ''){
							delete filterHash[i] ;
						}else{

							var exprClean = expr.replace(/\s*/g, '');
							filterHash[i] = new Object();
							filterHash[i]['raw'] = exprClean;
							if (/^[!<>=]/.test(expr)) {
								var oper = expr.match(/^([!<>=]+)/);
								var exp = exprClean.match(/^\W+([-]?\w+[.]?\w*)/i);
								filterHash[i]['operator'] = oper[0];
								if (exp === null){
									return;
								}
								filterHash[i]['expr'] = exp[1];

							}else{
								filterHash[i]['operator'] = '==';
								filterHash[i]['expr'] = new RegExp(`${exprClean}`,'i');
							}
						}

						tabFULLSPLIT.draw();
					
					});
					

			//button tab		
			var tabs = `<div class="tab" style="float: left;">
								<button id="FULLbutton" class="tablinks active" onclick="openTab(event, 'full')">COMPACT</button>
								<button id="FULLSPLITbutton" class="tablinks" onclick="openTab(event, 'fullsplit')">EXPANDED</button>
							</div>`;
			$('#tabFULLSPLIT_wrapper').prepend(tabs);
			

				//double click expand
				$('#tabFULLSPLIT tbody').on('dblclick', 'tr', function () {
					var rowID = this.id;			
					var fm = fullMode;
					if (rowID !== '' && fullMode === 'full'){

						if (dblClickMode == 'off'){
							dblClickMode = 'on';
							$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val(rowID);
							$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('' );
							filterHash[keyAnnotID] = new Object();
							filterHash[keyAnnotID]['raw'] = rowID;
							filterHash[keyAnnotID]['operator'] = '==';
							filterHash[keyAnnotID]['expr'] = new RegExp(`${filterHash[keyAnnotID]['raw']}`,'i');
							delete filterHash[keyType] ;

						}else{
							dblClickMode = 'off';
							$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('full' );
							$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val('');
							filterHash[keyType] = new Object();
							filterHash[keyType]['raw'] = 'full';
							filterHash[keyType]['operator'] = '==';
							filterHash[keyType]['expr'] = new RegExp(`${filterHash[keyType]['raw']}`,'i');
							delete filterHash[keyAnnotID] ;

						}
						
						tabFULLSPLIT.draw();

					}
				} ); //END DOUBLE CLICK

				//FILTERING FUNCTION PUSH AT INITIALISATION
				$.fn.dataTableExt.afnFiltering.push(
					function( oSettings, aData, iDataIndex ) {
						var filtHash = filterHash;
						var filtBool = true;
						for (var keys in filtHash){
							if (filtHash.hasOwnProperty(keys)) {
								var row_data = aData[keys];
			
								switch (filtHash[keys]['operator']){
									case '>': if(row_data > filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '<': if(row_data < filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '>=': if(row_data >= filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '<=': if(row_data <= filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '!=': if(row_data != filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '<>': if(row_data != filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '=': if(row_data == filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '==': var m = row_data.match(filtHash[keys]['expr']); if(m === null){return false;}else{filtBool = true;continue;}
								}
							}
						}
			
						return filtBool;
					}); //END function filterByExp




				//Restore state				
				var state = tabFULLSPLIT.state.loaded();						 
				if ( state  ) {
					//console.log('state__');
					for (var keys in filterHash){
						if (filterHash.hasOwnProperty(keys)) {
							$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keys+') input', $('.tooltipHeader td')[keys] ).val( filterHash[keys]['raw'] );
							if (filterHash[keys]['operator'] == '=='){
								filterHash[keys]['expr'] = new RegExp(`${filterHash[keys]['raw']}`,'i');
							}
						}
					}
					if (fullMode == 'fullsplit'){
						$('#FULLbutton').removeClass('active');
						$('#FULLSPLITbutton').addClass('active');
                                        }
					if (dblClickMode == 'on'){
						$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val(filterHash[keyAnnotID]['raw']);
					}

					
					
					tabFULLSPLIT.draw();
				}else{  //END restore
					
					//FULL view on load	
					window.onload =	$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('full' ).change();			
				}


		


	//click full split button
	openTab =	function(evt, cityName) {
				
				if(cityName === 'full'){
				
					fullMode = 'full';

					filterHash[keyType] = new Object();
					filterHash[keyType]['raw'] = cityName;
					filterHash[keyType]['operator'] = '==';
					filterHash[keyType]['expr'] = new RegExp(`${filterHash[keyType]['raw']}`,'i');
					
					$('#FULLSPLITbutton').removeClass('active');
					$('#FULLbutton').addClass('active');
					
					$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val(cityName );
					
				}else{
					
					fullMode = 'fullsplit';

					delete filterHash[keyType] ;
					$('#FULLbutton').removeClass('active');
					$('#FULLSPLITbutton').addClass('active');
					$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('');

				}
				
				if (dblClickMode == 'on'){
					dblClickMode = 'off';
					$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val('');
					delete filterHash[keyAnnotID] ;
				}
	
				
				$('#tabFULLSPLIT').DataTable().draw();
			}


			



});  //END document ready




</script>

<link rel="stylesheet" type="text/css" href='https://cdn.datatables.net/1.10.22/css/jquery.dataTables.min.css'>

<link rel="stylesheet" type="text/css" href='https://cdn.datatables.net/fixedcolumns/3.3.2/css/fixedColumns.dataTables.min.css'>


<style>
/* Style the tab */
	.tab {
		overflow: hidden;
		border: 1px solid #ccc;
		background-color: #f1f1f1;
		left: 0;
		bottom: 0;
		height: 27px;
		position: fixed;
		position: -webkit-sticky;
		position: sticky;
		margin-right: 10px;
		}

/* Style the buttons inside the tab */
	.tab button {
		background-color: inherit;
		float: left;
		border: none;
		outline: none;
		cursor: pointer;
		transition: 0.3s;
		font-size: 90%;
		height: 27px;
		}
/* Change background color of buttons on hover */
	.tab button:hover {
		background-color: #ddd;
		}

/* Create an active/current tablink class */
	.tab button.active {
		background-color: #ccc;
		}
	.tab button:focus {
		background-color: #ccc;
		}

	td {
		text-align: center;
		}
	thead input {
		width: 100%;
		}

	#tabFULLSPLIT.display td.sorting_1{
		background-color: inherit;
	}

	table.dataTable.display tbody tr.odd > .sorting_1, table.dataTable.order-column.stripe tbody tr.odd > .sorting_1{
		background-color: inherit; 
	}
	table.dataTable.display tbody tr.even > .sorting_1, table.dataTable.order-column.stripe tbody tr.even > .sorting_1{
		background-color: inherit; 
	}



/* Style the tab content */
	.tabcontent {
		display: block;
		padding: 6px 12px;
		/*border: 1px solid #ccc;*/
		border-top: none;
		}

	.tooltipHeader{
		position: relative;
		font-size: 90%;
		cursor: help;
	}		
	.tooltipHeader .tooltiptext{
		visibility: hidden;
		width: auto;
		min-width: 250px;
		max-width: 800px;
		height: auto;
		background-color: #1e1e1e;
		color: #fff;
		text-align: left;
		border-radius: 6px;
		padding: 5px 5px;
		position: absolute;
		z-index: 1;
		top: -5px;
		opacity: 0;
		transition: opacity 0.3s;
		text-overflow: ellipsis;
		white-space: normal;
		overflow-wrap: break-word;
		}
	.tooltipHeader:hover .tooltiptext{
		visibility: visible;
		opacity: 1;
	}	

	.tooltip {
		position: relative;
		display: inline-block;
		/*border-bottom: 1px dotted black;*/
		min-width: 100px;
		max-width: 300px;
		overflow-wrap: break-word;
		font-size: 70%;
		}
	
	.tooltip .tooltiptext {
		visibility: hidden;
		width: auto;
		min-width: 250px;
		max-width: 290px;
		max-height: 469px;
		background-color: #1e1e1e;
		color: #fff;
		text-align: left;
		border-radius: 6px;
		padding: 5px 5px;
		position: absolute;
		z-index: 1;
		top: 100%;
		opacity: 0;
		transition: opacity 0.3s;
		text-overflow: ellipsis;
		white-space: normal;
		overflow-wrap: break-word;
		font-size: 100%;
		overflow-y: scroll;
		}

	.tooltip:hover .tooltiptext {
		visibility: visible;
		opacity: 1;
		}
		
	.commentTitle {
		color: #FF69B4; 
		font-weight: bold;
	}


	div.dataTables_wrapper {
		width: 100%;
	/*	margin: 0 auto;*/
	}
	.DTFC_LeftBodyLiner { overflow-x: hidden; }
	.DTFC_RightBodyLiner { overflow-x: hidden; }





</style>


</head>

	<body>

<div> <h2>short.annotated___hg19</h2></div><div id="FULL+SPLIT" class="tabcontent">
	<table id='tabFULLSPLIT' class='display compact' >
        
		<thead>
			<tr>	<th class="tooltipHeader" >AnnotSV_ID<span style="left: 1%;max-width:250px" class="tooltiptext tooltip-bottom">AnnotSV identifier</span> 	</th>
	<th class="tooltipHeader" >ACMG_class<span style="left: 75%" class="tooltiptext tooltip-bottom">SV ranking class (from 1 to 5) following the 2020 joint consensus recommendation of ACMG and ClinGen</span> 	</th>
	<th class="tooltipHeader" >SV_type<span style="left: 62%" class="tooltiptext tooltip-bottom">Type of the SV (DEL, DUP...)</span> 	</th>
	<th class="tooltipHeader" >Annotation_mode<span style="left: 50%" class="tooltiptext tooltip-bottom">Type of annotation generated:<br> - annotation on the SV full length (full)<br> - annotation on each gene overlapped by the SV (split)</span> 	</th>
	<th class="tooltipHeader" >Gene_name<span style="left: 37%" class="tooltiptext tooltip-bottom">Gene symbol</span> 	</th>
	<th class="tooltipHeader" >Location<span style="left: 25%" class="tooltiptext tooltip-bottom">SV location in the gene’s<br> Values: txStart, txEnd, exon’i’, intron’i’ <br> e.g. « txStart-exon3 »</span> 	</th>
	<th class="tooltipHeader" >OMIM phenotype<span style="left: 12%" class="tooltiptext tooltip-bottom">e.g. Charcot-Marie-Tooth disease</span> 	</th>
	<th class="tooltipHeader" >Exomiser score<span style="left: 0%" class="tooltiptext tooltip-bottom">Exomiser score showing how close each overlapped gene is to the patient phenotype<br> Consistent phenotype: score > 0.5<br> Highly specific and consistent phenotype: score > 0.7</span> 	</th>
	<th class="tooltipHeader" >Regulatory elements<span style="right: 12%" class="tooltiptext tooltip-bottom">Name of the genes regulated by a regulatory element overlapped with the SV to annotate.</span> 	</th>
	<th class="tooltipHeader">Benign SV sources	</th>
	<th class="tooltipHeader">Benign SV sources	</th>
	<th class="tooltipHeader">Pathogenic SV sources	</th>
	<th class="tooltipHeader">Pathogenic SV sources	</th>
	<th class="tooltipHeader">Number of pathogenic SNV/indel overlapped	</th>
	<th class="tooltipHeader" >ENCODE blacklist characteristics left<span style="right: 87%" class="tooltiptext tooltip-bottom">ENCODE blacklist regions characteristics around the left SV breakpoint (+/- 100bp)</span> 	</th>
	<th class="tooltipHeader" >ENCODE blacklist characteristics right<span style="right: 100%" class="tooltiptext tooltip-bottom">ENCODE blacklist regions characteristics around the right SV breakpoint (+/- 100bp)</span> 	</th>

			</tr>
		</thead>
	<tbody>
<tr id="1_174417588_191888383_DEL_1" class="full" style="background-color:#F7D8DD" >
	<td data-order="1" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795<br><span class="commentTitle">Gene_count :</span> 107</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="full"><div class="tooltip">full<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> full</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="ABL2; ACBD6; ANGPTL1; APOBEC4; ARPC5; ASTN1; AXDND1; BRINP2; BRINP3; C1orf21; C1orf220; CACNA1E; CACYBP; CEP350; COLGALT2; COP1; CRYZL2P; CRYZL2P-SEC16B; DHX9; EDEM3; FAM163A; FAM20B; GLUL; GPR52; GS1-279B7.1; HMCN1; IER5; IVNS1ABP; KIAA0040; KIAA1614; KIAA1614-AS1; LAMC1; LAMC1-AS1; LAMC2; LHX4; LHX4-AS1; LINC00272; LINC01036; LINC01037; LINC01344; LINC01350; LINC01351; LINC01633; LINC01645; LINC01657; LINC01680; LINC01686; LINC01688; LINC01699; LINC01720; LINC01732; LINC01741; LOC647070; MIR12116; MIR1843; MIR3121; MIR4424; MIR488; MR1; MRPS14; NCF2; NIBAN1; NMNAT2; NPHS2; NPL; ODR4; OVAAL; PACERR; PAPPA2; PDC; PDC-AS1; PLA2G4A; PRG4; PTGS2; QSOX1; RABGAP1L; RABGAP1L-AS1; RALGPS2; RASAL2; RASAL2-AS1; RGL1; RGS16; RGS8; RGSL1; RNASEL; RNF2; SCARNA3; SEC16B; SHCBP1L; SMG7; SMG7-AS1; SOAT1; STX6; SWT1; TDRD5; TEDDM1; TEX35; TNN; TNR; TOR1AIP1; TOR1AIP2; TOR3A; TPR; TRMT1L; TSEN15; XPR1; ZNF648"><div class="tooltip">ABL2; ACBD6; ANGPTL1; APOBEC4; ARPC5; ASTN1; [...]<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ABL2; ACBD6; ANGPTL1; APOBEC4; ARPC5; ASTN1; AXDND1; BRINP2; BRINP3; C1orf21; C1orf220; CACNA1E; CACYBP; CEP350; COLGALT2; COP1; CRYZL2P; CRYZL2P-SEC16B; DHX9; EDEM3; FAM163A; FAM20B; GLUL; GPR52; GS1-279B7.1; HMCN1; IER5; IVNS1ABP; KIAA0040; KIAA1614; KIAA1614-AS1; LAMC1; LAMC1-AS1; LAMC2; LHX4; LHX4-AS1; LINC00272; LINC01036; LINC01037; LINC01344; LINC01350; LINC01351; LINC01633; LINC01645; LINC01657; LINC01680; LINC01686; LINC01688; LINC01699; LINC01720; LINC01732; LINC01741; LOC647070; MIR12116; MIR1843; MIR3121; MIR4424; MIR488; MR1; MRPS14; NCF2; NIBAN1; NMNAT2; NPHS2; NPL; ODR4; OVAAL; PACERR; PAPPA2; PDC; PDC-AS1; PLA2G4A; PRG4; PTGS2; QSOX1; RABGAP1L; RABGAP1L-AS1; RALGPS2; RASAL2; RASAL2-AS1; RGL1; RGS16; RGS8; RGSL1; RNASEL; RNF2; SCARNA3; SEC16B; SHCBP1L; SMG7; SMG7-AS1; SOAT1; STX6; SWT1; TDRD5; TEDDM1; TEX35; TNN; TNR; TOR1AIP1; TOR1AIP2; TOR3A; TPR; TRMT1L; TSEN15; XPR1; ZNF648<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> .<br><span class="commentTitle">Overlapped_tx_length :</span> .<br><span class="commentTitle">Overlapped_CDS_length :</span> .<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="ANKRD45; DARS2 (HI=30; TS=0); GAS5; GAS5-AS1; KLHL20; RABGAP1L-DT; RC3H1; RGS18; SERPINC1; SNORA67; ZBTB37"><div class="tooltip">ANKRD45; DARS2 (HI=30; TS=0); GAS5; GAS5-AS1;[...]<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> ANKRD45; DARS2 (HI=30; TS=0); GAS5; GAS5-AS1; KLHL20; RABGAP1L-DT; RC3H1; RGS18; SERPINC1; SNORA67; ZBTB37</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:174982093-174992591; 1:179519673-179545083; 1:179851419-179889211; 1:180199398-180248119; 1:180199433-180244188; 1:180601157-180859415; 1:181452652-181777219; 1:182347232-182360925; 1:182542768-182558391; 1:183155398-183214262; 1:183524696-183560049; 1:184020863-184066619; 1:185703522-186160081; 1:186265410-186283697; 1:186412697-186430246; 1:186798080-186958113"><div class="tooltip">1:174982093-174992591; 1:179519673-179545083;[...]<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:174982093-174992591; 1:179519673-179545083; 1:179851419-179889211; 1:180199398-180248119; 1:180199433-180244188; 1:180601157-180859415; 1:181452652-181777219; 1:182347232-182360925; 1:182542768-182558391; 1:183155398-183214262; 1:183524696-183560049; 1:184020863-184066619; 1:185703522-186160081; 1:186265410-186283697; 1:186412697-186430246; 1:186798080-186958113<span class="commentTitle">P_loss_coord :</span> 1:174982093-174992591; 1:179519673-179545083; 1:179851419-179889211; 1:180199398-180248119; 1:180199433-180244188; 1:180601157-180859415; 1:181452652-181777219; 1:182347232-182360925; 1:182542768-182558391; 1:183155398-183214262; 1:183524696-183560049; 1:184020863-184066619; 1:185703522-186160081; 1:186265410-186283697; 1:186412697-186430246; 1:186798080-186958113<br><span class="commentTitle">P_loss_source :</span> HI3:LHX4; morbid:CACNA1E; morbid:GLUL; morbid:HMCN1; morbid:LAMC2; morbid:LHX4; morbid:MRPS14; morbid:NCF2; morbid:NPHS2; morbid:PDC; morbid:PLA2G4A; morbid:PRG4; morbid:RNASEL; morbid:TOR1AIP1; morbid:TSEN15; morbid:XPR1<br><span class="commentTitle">P_loss_phen :</span> "?Combined oxidative phosphorylation deficiency 38, 618378 (3); ?Muscular dystrophy, autosomal recessive, with rigid spine and distal joint contractures, 617072 (3); Basal ganglia calcification, idiopathic, 6, 616413 (3); Camptodactyly-arthropathy-coxa vara-pericarditis syndrome, 208250 (3); Chronic granulomatous disease due to deficiency of NCF-2, 233710 (3); Epidermolysis bullosa, junctional, Herlitz type, 226700 (3); Epidermolysis bullosa, junctional, non-Herlitz type, 226650 (3); Epileptic encephalopathy, early infantile, 69, 618285 (3); Gastrointestinal ulceration, recurrent, with dysfunctional platelets, 618372 (3); Glutamine deficiency, congenital, 610015 (3); Macular degeneration, age-related, 1, 603075 (3); Nephrotic syndrome, type 2, 600995 (3); Paroxysmal nonkinesigenic dyskinesia 1, 118800 (3); Pituitary hormone deficiency, combined, 4, 262700 (3); Pontocerebellar hypoplasia, type 2F, 617026 (3); Prostate cancer 1, 601518 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:174982093-174992591; 1:179519673-179545083; 1:179851419-179889211; 1:180199398-180248119; 1:180199433-180244188; 1:180601157-180859415; 1:181452652-181777219; 1:182347232-182360925; 1:182542768-182558391; 1:183155398-183214262; 1:183524696-183560049; 1:184020863-184066619; 1:185703522-186160081; 1:186265410-186283697; 1:186412697-186430246; 1:186798080-186958113"><div class="tooltip">1:174982093-174992591; 1:179519673-179545083;[...]<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:174982093-174992591; 1:179519673-179545083; 1:179851419-179889211; 1:180199398-180248119; 1:180199433-180244188; 1:180601157-180859415; 1:181452652-181777219; 1:182347232-182360925; 1:182542768-182558391; 1:183155398-183214262; 1:183524696-183560049; 1:184020863-184066619; 1:185703522-186160081; 1:186265410-186283697; 1:186412697-186430246; 1:186798080-186958113<span class="commentTitle">P_loss_coord :</span> 1:174982093-174992591; 1:179519673-179545083; 1:179851419-179889211; 1:180199398-180248119; 1:180199433-180244188; 1:180601157-180859415; 1:181452652-181777219; 1:182347232-182360925; 1:182542768-182558391; 1:183155398-183214262; 1:183524696-183560049; 1:184020863-184066619; 1:185703522-186160081; 1:186265410-186283697; 1:186412697-186430246; 1:186798080-186958113</span></div>	</td>
	<td data-search="78"><div class="tooltip">78<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 78<span class="commentTitle">P_snvindel_nb :</span> 78<br><span class="commentTitle">P_snvindel_phen :</span> "Camptodactyly-arthropathy-coxa_vara-pericarditis_syndrome; Chronic_granulomatous_disease,_autosomal_recessive_cytochrome_b-positive,_type_2; Chronic_kidney_disease; DEVELOPMENTAL_AND_EPILEPTIC_ENCEPHALOPATHY_69; GASTROINTESTINAL_ULCERATION,_RECURRENT,_WITH_DYSFUNCTIONAL_PLATELETS; Hereditary_nephrotic_syndrome; Idiopathic_nephrotic_syndrome; Inborn_genetic_diseases; Junctional_epidermolysis_bullosa; Junctional_epidermolysis_bullosa,_non-Herlitz_type; Junctional_epidermolysis_bullosa_gravis_of_Herlitz; Muscular_dystrophy,_limb-girdle,_type_2y; Nephrotic_range_proteinuria; Nephrotic_syndrome,_idiopathic,_steroid-resistant; Nephrotic_syndrome,_type_2,_susceptibility_to; Non-progressive_neurodevelopmental_disorder_with_spasticity_and_transient_opisthotonus; Proteinuria; Short_stature-pituitary_and_cerebellar_defects-small_sella_turcica_syndrome"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .<br><span class="commentTitle">ENCODE_blacklist_left :</span> .<br><span class="commentTitle">SegDup_left :</span> .<br><span class="commentTitle">Repeat_coord_left :</span> .<br><span class="commentTitle">Repeat_type_left :</span> .<br><span class="commentTitle">Gap_left :</span> .<br><span class="commentTitle">GC_content_left :</span> 0.440</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .<br><span class="commentTitle">ENCODE_blacklist_right :</span> .<br><span class="commentTitle">SegDup_right :</span> .<br><span class="commentTitle">Repeat_coord_right :</span> .<br><span class="commentTitle">Repeat_type_right :</span> .<br><span class="commentTitle">Gap_right :</span> .<br><span class="commentTitle">GC_content_right :</span> 0.325</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="2" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LAMC1-AS1"><div class="tooltip">LAMC1-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LAMC1-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_149048<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 2868<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="3" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01037"><div class="tooltip">LINC01037<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01037<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_132374<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 33595<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="4" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="COP1"><div class="tooltip">COP1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> COP1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_022457<br><span class="commentTitle">Exon_count :</span> 20<br><span class="commentTitle">Overlapped_tx_length :</span> 262456<br><span class="commentTitle">Overlapped_CDS_length :</span> 2196<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="5" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LHX4-AS1"><div class="tooltip">LHX4-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LHX4-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_037642<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 5019<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="1"><div class="tooltip">1<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 1<span class="commentTitle">P_snvindel_nb :</span> 1<br><span class="commentTitle">P_snvindel_phen :</span> Short_stature-pituitary_and_cerebellar_defects-small_sella_turcica_syndrome</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="6" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="CRYZL2P"><div class="tooltip">CRYZL2P<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CRYZL2P<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_037167<br><span class="commentTitle">Exon_count :</span> 10<br><span class="commentTitle">Overlapped_tx_length :</span> 31868<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="7" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC00272"><div class="tooltip">LINC00272<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC00272<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_034131<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 7193<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="8" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="KIAA1614-AS1"><div class="tooltip">KIAA1614-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KIAA1614-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_145481<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 5197<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="9" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="GS1-279B7.1"><div class="tooltip">GS1-279B7.1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> GS1-279B7.1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_038424<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 11193<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="10" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01036"><div class="tooltip">LINC01036<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01036<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_126347<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 267403<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="11" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="CRYZL2P-SEC16B"><div class="tooltip">CRYZL2P-SEC16B<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CRYZL2P-SEC16B<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001356505<br><span class="commentTitle">Exon_count :</span> 34<br><span class="commentTitle">Overlapped_tx_length :</span> 109219<br><span class="commentTitle">Overlapped_CDS_length :</span> 3186<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="12" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="C1orf21"><div class="tooltip">C1orf21<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C1orf21<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "8,92E+03"<br><span class="commentTitle">ExAC_pLI :</span> "8,19E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 11.06<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.146583806990107<br><span class="commentTitle">ExAC_delZ :</span> 0.540202541611708<br><span class="commentTitle">ExAC_dupZ :</span> -0.152571183969044<br><span class="commentTitle">ExAC_synZ :</span> -0.328517091593928<br><span class="commentTitle">ExAC_misZ :</span> 0.929289408725248<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_030806<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 241991<br><span class="commentTitle">Overlapped_CDS_length :</span> 366<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="13" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="CACYBP"><div class="tooltip">CACYBP<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CACYBP<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "9,20E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,43E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 10.7<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.100649792319943<br><span class="commentTitle">ExAC_delZ :</span> -0.50180352431752<br><span class="commentTitle">ExAC_dupZ :</span> 0.113246057609504<br><span class="commentTitle">ExAC_synZ :</span> -0.662989654179239<br><span class="commentTitle">ExAC_misZ :</span> 0.959177412664334<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_014412<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 12081<br><span class="commentTitle">Overlapped_CDS_length :</span> 687<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="14" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="IVNS1ABP"><div class="tooltip">IVNS1ABP<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> IVNS1ABP<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "9,94E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,99E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 19.32<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.394746602170845<br><span class="commentTitle">ExAC_delZ :</span> -0.391999193870284<br><span class="commentTitle">ExAC_dupZ :</span> 0.83273043709851<br><span class="commentTitle">ExAC_synZ :</span> 789.407.776<br><span class="commentTitle">ExAC_misZ :</span> 208.443.737<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_006469<br><span class="commentTitle">Exon_count :</span> 15<br><span class="commentTitle">Overlapped_tx_length :</span> 20856<br><span class="commentTitle">Overlapped_CDS_length :</span> 1929<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Immunodeficiency 70, 618969 (3)""><div class="tooltip">"Immunodeficiency 70, 618969 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Immunodeficiency 70, 618969 (3)"<br><span class="commentTitle">OMIM_ID :</span> 609209<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="15" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="IER5"><div class="tooltip">IER5<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> IER5<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "8,69E+03"<br><span class="commentTitle">ExAC_pLI :</span> "6,66E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 72.53<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 2.089.537.939<br><span class="commentTitle">ExAC_misZ :</span> -672.143.891<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_016545<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 4201<br><span class="commentTitle">Overlapped_CDS_length :</span> 984<br><span class="commentTitle">Overlapped_CDS_percent :</span> 24<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="16" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="ABL2"><div class="tooltip">ABL2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ABL2<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "3,64E-03"<br><span class="commentTitle">ExAC_pLI :</span> "9,70E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 24.57<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.971116167575072<br><span class="commentTitle">ExAC_delZ :</span> 0.312084882500471<br><span class="commentTitle">ExAC_dupZ :</span> 29.449.819<br><span class="commentTitle">ExAC_synZ :</span> 1.503.647.868<br><span class="commentTitle">ExAC_misZ :</span> 113.870.319<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_007314<br><span class="commentTitle">Exon_count :</span> 12<br><span class="commentTitle">Overlapped_tx_length :</span> 130348<br><span class="commentTitle">Overlapped_CDS_length :</span> 3549<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="17" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="HMCN1"><div class="tooltip">HMCN1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HMCN1<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "3,63E-34"<br><span class="commentTitle">ExAC_pLI :</span> "2,19E-14"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 29.46<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.107520212796412<br><span class="commentTitle">ExAC_delZ :</span> 1.271.430.451<br><span class="commentTitle">ExAC_dupZ :</span> 0.730008216027874<br><span class="commentTitle">ExAC_synZ :</span> -0.847528083602207<br><span class="commentTitle">ExAC_misZ :</span> -1.097.513.255<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_031935<br><span class="commentTitle">Exon_count :</span> 107<br><span class="commentTitle">Overlapped_tx_length :</span> 456559<br><span class="commentTitle">Overlapped_CDS_length :</span> 16908<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""(Macular degeneration, age-related, 1), 603075 (3)""><div class="tooltip">"(Macular degeneration, age-related, 1), 6030[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "(Macular degeneration, age-related, 1), 603075 (3)"<br><span class="commentTitle">OMIM_ID :</span> 608548<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:185703522-186160081"><div class="tooltip">1:185703522-186160081<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:185703522-186160081<span class="commentTitle">P_loss_coord :</span> 1:185703522-186160081<br><span class="commentTitle">P_loss_source :</span> morbid:HMCN1<br><span class="commentTitle">P_loss_phen :</span> "Macular degeneration, age-related, 1, 603075 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:185703522-186160081"><div class="tooltip">1:185703522-186160081<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:185703522-186160081<span class="commentTitle">P_loss_coord :</span> 1:185703522-186160081</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="18" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="ARPC5"><div class="tooltip">ARPC5<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ARPC5<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "8,22E+03"<br><span class="commentTitle">ExAC_pLI :</span> "7,73E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 29.75<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.571460617600802<br><span class="commentTitle">ExAC_delZ :</span> 0.705139884844296<br><span class="commentTitle">ExAC_dupZ :</span> 0.297482496834465<br><span class="commentTitle">ExAC_synZ :</span> 0.907970582843996<br><span class="commentTitle">ExAC_misZ :</span> -1.626.839.212<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001270439<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 14938<br><span class="commentTitle">Overlapped_CDS_length :</span> 465<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="19" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="BRINP3"><div class="tooltip">BRINP3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> BRINP3<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "1,46E+03"<br><span class="commentTitle">ExAC_pLI :</span> "3,76E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 16.41<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.0118811869684838<br><span class="commentTitle">ExAC_delZ :</span> -0.361633386376747<br><span class="commentTitle">ExAC_dupZ :</span> 0.165152871455676<br><span class="commentTitle">ExAC_synZ :</span> 0.0384664835385234<br><span class="commentTitle">ExAC_misZ :</span> 0.689454037657182<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_199051<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 380206<br><span class="commentTitle">Overlapped_CDS_length :</span> 2301<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="20" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="EDEM3"><div class="tooltip">EDEM3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> EDEM3<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "2,47E+01"<br><span class="commentTitle">ExAC_pLI :</span> "1,23E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 20.92<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.14846317503859<br><span class="commentTitle">ExAC_delZ :</span> 0.452360278680309<br><span class="commentTitle">ExAC_dupZ :</span> -0.45583936804905<br><span class="commentTitle">ExAC_synZ :</span> 0.546989853190578<br><span class="commentTitle">ExAC_misZ :</span> -0.212935611987177<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001319960<br><span class="commentTitle">Exon_count :</span> 21<br><span class="commentTitle">Overlapped_tx_length :</span> 64621<br><span class="commentTitle">Overlapped_CDS_length :</span> 2847<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="21" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="LHX4"><div class="tooltip">LHX4<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LHX4<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "1,75E+02"<br><span class="commentTitle">ExAC_pLI :</span> "4,01E+03"<br><span class="commentTitle">HI :</span> 3<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 6.37<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.876916354070022<br><span class="commentTitle">ExAC_delZ :</span> 0.216275564918622<br><span class="commentTitle">ExAC_dupZ :</span> -201.920.952<br><span class="commentTitle">ExAC_synZ :</span> 0.525077070941348<br><span class="commentTitle">ExAC_misZ :</span> 0.77491523731198<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> monoallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> LHX4-RELATED COMBINED PITUITARY HORMONE DEFICIENCY<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_033343<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 48721<br><span class="commentTitle">Overlapped_CDS_length :</span> 1173<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Pituitary hormone deficiency, combined, 4, 262700 (3)""><div class="tooltip">"Pituitary hormone deficiency, combined, 4, 2[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Pituitary hormone deficiency, combined, 4, 262700 (3)"<br><span class="commentTitle">OMIM_ID :</span> 602146<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:180199398-180248119; 1:180199433-180244188"><div class="tooltip">1:180199398-180248119; 1:180199433-180244188<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:180199398-180248119; 1:180199433-180244188<span class="commentTitle">P_loss_coord :</span> 1:180199398-180248119; 1:180199433-180244188<br><span class="commentTitle">P_loss_source :</span> HI3:LHX4; morbid:LHX4<br><span class="commentTitle">P_loss_phen :</span> "Pituitary hormone deficiency, combined, 4, 262700 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:180199398-180248119; 1:180199433-180244188"><div class="tooltip">1:180199398-180248119; 1:180199433-180244188<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:180199398-180248119; 1:180199433-180244188<span class="commentTitle">P_loss_coord :</span> 1:180199398-180248119; 1:180199433-180244188</span></div>	</td>
	<td data-search="2"><div class="tooltip">2<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 2<span class="commentTitle">P_snvindel_nb :</span> 2<br><span class="commentTitle">P_snvindel_phen :</span> Short_stature-pituitary_and_cerebellar_defects-small_sella_turcica_syndrome</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="22" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="LAMC2"><div class="tooltip">LAMC2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LAMC2<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "2,88E-08"<br><span class="commentTitle">ExAC_pLI :</span> "7,93E-01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 41.89<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 376.229.736<br><span class="commentTitle">ExAC_delZ :</span> 0.836680837273556<br><span class="commentTitle">ExAC_dupZ :</span> -1.547.909.360<br><span class="commentTitle">ExAC_synZ :</span> -0.554966249844879<br><span class="commentTitle">ExAC_misZ :</span> -0.493808536220432<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_005562<br><span class="commentTitle">Exon_count :</span> 23<br><span class="commentTitle">Overlapped_tx_length :</span> 58864<br><span class="commentTitle">Overlapped_CDS_length :</span> 3582<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Epidermolysis bullosa, junctional, non-Herlitz type, 226650 (3)/  Epidermolysis bullosa, junctional, Herlitz type, 226700 (3)""><div class="tooltip">"Epidermolysis bullosa, junctional, non-Herli[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Epidermolysis bullosa, junctional, non-Herlitz type, 226650 (3)/  Epidermolysis bullosa, junctional, Herlitz type, 226700 (3)"<br><span class="commentTitle">OMIM_ID :</span> 150292<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:183155398-183214262"><div class="tooltip">1:183155398-183214262<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:183155398-183214262<span class="commentTitle">P_loss_coord :</span> 1:183155398-183214262<br><span class="commentTitle">P_loss_source :</span> morbid:LAMC2<br><span class="commentTitle">P_loss_phen :</span> "Epidermolysis bullosa, junctional, Herlitz type, 226700 (3); Epidermolysis bullosa, junctional, non-Herlitz type, 226650 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:183155398-183214262"><div class="tooltip">1:183155398-183214262<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:183155398-183214262<span class="commentTitle">P_loss_coord :</span> 1:183155398-183214262</span></div>	</td>
	<td data-search="22"><div class="tooltip">22<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 22<span class="commentTitle">P_snvindel_nb :</span> 22<br><span class="commentTitle">P_snvindel_phen :</span> "Junctional_epidermolysis_bullosa; Junctional_epidermolysis_bullosa,_non-Herlitz_type; Junctional_epidermolysis_bullosa_gravis_of_Herlitz"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="23" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="AXDND1"><div class="tooltip">AXDND1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> AXDND1<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "4,31E-22"<br><span class="commentTitle">ExAC_pLI :</span> "1,80E-11"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 75.2<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.72470784051635<br><span class="commentTitle">ExAC_delZ :</span> -0.45032597997007<br><span class="commentTitle">ExAC_dupZ :</span> -0.910013642703016<br><span class="commentTitle">ExAC_synZ :</span> -0.398504545280016<br><span class="commentTitle">ExAC_misZ :</span> -875.875.796<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_144696<br><span class="commentTitle">Exon_count :</span> 26<br><span class="commentTitle">Overlapped_tx_length :</span> 188765<br><span class="commentTitle">Overlapped_CDS_length :</span> 3039<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="9"><div class="tooltip">9<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 9<span class="commentTitle">P_snvindel_nb :</span> 9<br><span class="commentTitle">P_snvindel_phen :</span> "Chronic_kidney_disease; Idiopathic_nephrotic_syndrome; Nephrotic_range_proteinuria; Nephrotic_syndrome,_idiopathic,_steroid-resistant; Nephrotic_syndrome,_type_2,_susceptibility_to"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="24" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background: linear-gradient(-45deg,#FFFF00 50%, white 50% )" data-search="GPR52"><div class="tooltip">GPR52<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> GPR52<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "9,54E+01"<br><span class="commentTitle">ExAC_pLI :</span> "6,55E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 18.14<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.152215041750649<br><span class="commentTitle">ExAC_delZ :</span> -0.0905395070419039<br><span class="commentTitle">ExAC_dupZ :</span> 0.303852135642258<br><span class="commentTitle">ExAC_synZ :</span> 0.236220523022167<br><span class="commentTitle">ExAC_misZ :</span> 0.963697307791825<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="exon1-txEnd"><div class="tooltip">exon1-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> exon1-txEnd<br><span class="commentTitle">Location2 :</span> CDS-3'UTR<br><span class="commentTitle">Tx :</span> NM_005684<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 1095<br><span class="commentTitle">Overlapped_CDS_length :</span> 747<br><span class="commentTitle">Overlapped_CDS_percent :</span> 52<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> 5'<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="25" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="KIAA1614"><div class="tooltip">KIAA1614<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KIAA1614<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,15E-11"<br><span class="commentTitle">ExAC_pLI :</span> "2,19E-09"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 87.16<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.544346543386391<br><span class="commentTitle">ExAC_delZ :</span> -0.12284442689439<br><span class="commentTitle">ExAC_dupZ :</span> 0.771774715990711<br><span class="commentTitle">ExAC_synZ :</span> 0.486722148153725<br><span class="commentTitle">ExAC_misZ :</span> 0.482389405224916<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_020950<br><span class="commentTitle">Exon_count :</span> 9<br><span class="commentTitle">Overlapped_tx_length :</span> 38718<br><span class="commentTitle">Overlapped_CDS_length :</span> 3573<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="26" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="COLGALT2"><div class="tooltip">COLGALT2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> COLGALT2<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,09E-06"<br><span class="commentTitle">ExAC_pLI :</span> "5,02E-03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 41.43<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.605748881512383<br><span class="commentTitle">ExAC_delZ :</span> 0.999791700720404<br><span class="commentTitle">ExAC_dupZ :</span> 0.240267322845379<br><span class="commentTitle">ExAC_synZ :</span> 0.160385190174272<br><span class="commentTitle">ExAC_misZ :</span> 0.906800590576741<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_015101<br><span class="commentTitle">Exon_count :</span> 12<br><span class="commentTitle">Overlapped_tx_length :</span> 101894<br><span class="commentTitle">Overlapped_CDS_length :</span> 1881<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="27" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="FAM20B"><div class="tooltip">FAM20B<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> FAM20B<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "3,89E-02"<br><span class="commentTitle">ExAC_pLI :</span> "2,86E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 17.51<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.840046256232829<br><span class="commentTitle">ExAC_delZ :</span> 0.171823096405637<br><span class="commentTitle">ExAC_dupZ :</span> 0.992116734846855<br><span class="commentTitle">ExAC_synZ :</span> 0.444738936665665<br><span class="commentTitle">ExAC_misZ :</span> 0.18159793942414<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001324310<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 50771<br><span class="commentTitle">Overlapped_CDS_length :</span> 1230<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="28" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="FAM163A"><div class="tooltip">FAM163A<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> FAM163A<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "1,16E+03"<br><span class="commentTitle">ExAC_pLI :</span> "5,80E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 47.42<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.233503983665655<br><span class="commentTitle">ExAC_delZ :</span> -0.452174816492047<br><span class="commentTitle">ExAC_dupZ :</span> 0.531546310251041<br><span class="commentTitle">ExAC_synZ :</span> -0.267089489739887<br><span class="commentTitle">ExAC_misZ :</span> 0.434876764965883<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001329713<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 88423<br><span class="commentTitle">Overlapped_CDS_length :</span> 504<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="29" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="ACBD6"><div class="tooltip">ACBD6<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ACBD6<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "2,26E-02"<br><span class="commentTitle">ExAC_pLI :</span> "6,50E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 22.84<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.018265252602563<br><span class="commentTitle">ExAC_delZ :</span> -0.159927915449793<br><span class="commentTitle">ExAC_dupZ :</span> 0.074176193145211<br><span class="commentTitle">ExAC_synZ :</span> -0.0712672100424217<br><span class="commentTitle">ExAC_misZ :</span> -0.300012422402704<br><span class="commentTitle">DDD_status :</span> possible<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> AUTOSOMAL RECESSIVE MENTAL RETARDATION<br><span class="commentTitle">DDD_pmid :</span> 21937992</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_032360<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 214348<br><span class="commentTitle">Overlapped_CDS_length :</span> 849<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="30" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="APOBEC4"><div class="tooltip">APOBEC4<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> APOBEC4<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "8,17E-01"<br><span class="commentTitle">ExAC_pLI :</span> "6,18E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 72.37<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.295638551196295<br><span class="commentTitle">ExAC_delZ :</span> 0.573157337034761<br><span class="commentTitle">ExAC_dupZ :</span> 0.0161626821626735<br><span class="commentTitle">ExAC_synZ :</span> 452.860.162<br><span class="commentTitle">ExAC_misZ :</span> -0.182611406726764<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_203454<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 7041<br><span class="commentTitle">Overlapped_CDS_length :</span> 1104<br><span class="commentTitle">Overlapped_CDS_percent :</span> 97<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="31" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="C1orf220"><div class="tooltip">C1orf220<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C1orf220<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "2,36E+03"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 99.62<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_033186<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 6094<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="32" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="ANGPTL1"><div class="tooltip">ANGPTL1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ANGPTL1<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "1,21E-06"<br><span class="commentTitle">ExAC_pLI :</span> "1,53E-04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 29.39<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.70115762350698<br><span class="commentTitle">ExAC_delZ :</span> 0.0700991326883787<br><span class="commentTitle">ExAC_dupZ :</span> 0.848439661232589<br><span class="commentTitle">ExAC_synZ :</span> -0.674776288292457<br><span class="commentTitle">ExAC_misZ :</span> -0.728725892974451<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001376763<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 21543<br><span class="commentTitle">Overlapped_CDS_length :</span> 1476<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="33" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="KIAA0040"><div class="tooltip">KIAA0040<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KIAA0040<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "2,01E+03"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 58.51<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001162893<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 36002<br><span class="commentTitle">Overlapped_CDS_length :</span> 300<br><span class="commentTitle">Overlapped_CDS_percent :</span> 69<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="34" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="ASTN1"><div class="tooltip">ASTN1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ASTN1<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "9,99E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 8.12<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -784.767.695<br><span class="commentTitle">ExAC_delZ :</span> 0.163977167029357<br><span class="commentTitle">ExAC_dupZ :</span> -834.717.333<br><span class="commentTitle">ExAC_synZ :</span> 0.299470830988838<br><span class="commentTitle">ExAC_misZ :</span> 1.930.211.769<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001364856<br><span class="commentTitle">Exon_count :</span> 23<br><span class="commentTitle">Overlapped_tx_length :</span> 303649<br><span class="commentTitle">Overlapped_CDS_length :</span> 3909<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="35" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="CEP350"><div class="tooltip">CEP350<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CEP350<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 36.97<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -2.141.343.579<br><span class="commentTitle">ExAC_delZ :</span> 0.586763826755621<br><span class="commentTitle">ExAC_dupZ :</span> 801.725.734<br><span class="commentTitle">ExAC_synZ :</span> -0.859348536431965<br><span class="commentTitle">ExAC_misZ :</span> 326.478.292<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_014810<br><span class="commentTitle">Exon_count :</span> 38<br><span class="commentTitle">Overlapped_tx_length :</span> 160066<br><span class="commentTitle">Overlapped_CDS_length :</span> 9354<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="36" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="GLUL"><div class="tooltip">GLUL<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> GLUL<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "9,96E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,78E+03"<br><span class="commentTitle">HI :</span> 30<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 9.54<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 1.003.908.920<br><span class="commentTitle">ExAC_delZ :</span> 0.952628702448049<br><span class="commentTitle">ExAC_dupZ :</span> 31.036.591<br><span class="commentTitle">ExAC_synZ :</span> -0.735865132853948<br><span class="commentTitle">ExAC_misZ :</span> 902.668.777<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> uncertain<br><span class="commentTitle">DDD_disease :</span> CONGENITAL SYSTEMIC GLUTAMINE DEFICIENCY<br><span class="commentTitle">DDD_pmid :</span> 21353613; 16267323</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_002065<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 14114<br><span class="commentTitle">Overlapped_CDS_length :</span> 1122<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Glutamine deficiency, congenital, 610015 (3)""><div class="tooltip">"Glutamine deficiency, congenital, 610015 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Glutamine deficiency, congenital, 610015 (3)"<br><span class="commentTitle">OMIM_ID :</span> 138290<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:182347232-182360925"><div class="tooltip">1:182347232-182360925<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:182347232-182360925<span class="commentTitle">P_loss_coord :</span> 1:182347232-182360925<br><span class="commentTitle">P_loss_source :</span> morbid:GLUL<br><span class="commentTitle">P_loss_phen :</span> "Glutamine deficiency, congenital, 610015 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:182347232-182360925"><div class="tooltip">1:182347232-182360925<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:182347232-182360925<span class="commentTitle">P_loss_coord :</span> 1:182347232-182360925</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="37" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="BRINP2"><div class="tooltip">BRINP2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> BRINP2<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "9,99E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,99E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 20.53<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.321233586213818<br><span class="commentTitle">ExAC_delZ :</span> -0.0700046572310053<br><span class="commentTitle">ExAC_dupZ :</span> 0.42502613687069<br><span class="commentTitle">ExAC_synZ :</span> -0.314124973874092<br><span class="commentTitle">ExAC_misZ :</span> 0.847761455283399<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_021165<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 111465<br><span class="commentTitle">Overlapped_CDS_length :</span> 2352<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="38" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="CACNA1E"><div class="tooltip">CACNA1E<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CACNA1E<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 13.63<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.694217030778858<br><span class="commentTitle">ExAC_delZ :</span> 0.675924085483259<br><span class="commentTitle">ExAC_dupZ :</span> 0.608685885768939<br><span class="commentTitle">ExAC_synZ :</span> -0.234528339988778<br><span class="commentTitle">ExAC_misZ :</span> -5.439.542<br><span class="commentTitle">DDD_status :</span> probable/ confirmed<br><span class="commentTitle">DDD_mode :</span> monoallelic/ monoallelic<br><span class="commentTitle">DDD_consequence :</span> gain of function/ activating<br><span class="commentTitle">DDD_disease :</span> "Epileptic Encephalopathy with Contractures, Macrocephaly, and Dyskinesia/ Developmental and Epileptic Encephalopathy with Contractures Macrocephaly and Dyskinesias"<br><span class="commentTitle">DDD_pmid :</span> 30343943/ 30849329</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001205293<br><span class="commentTitle">Exon_count :</span> 48<br><span class="commentTitle">Overlapped_tx_length :</span> 324567<br><span class="commentTitle">Overlapped_CDS_length :</span> 6942<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Epileptic encephalopathy, early infantile, 69, 618285 (3)""><div class="tooltip">"Epileptic encephalopathy, early infantile, 6[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Epileptic encephalopathy, early infantile, 69, 618285 (3)"<br><span class="commentTitle">OMIM_ID :</span> 601013<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:181452652-181777219"><div class="tooltip">1:181452652-181777219<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:181452652-181777219<span class="commentTitle">P_loss_coord :</span> 1:181452652-181777219<br><span class="commentTitle">P_loss_source :</span> morbid:CACNA1E<br><span class="commentTitle">P_loss_phen :</span> "Epileptic encephalopathy, early infantile, 69, 618285 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:181452652-181777219"><div class="tooltip">1:181452652-181777219<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:181452652-181777219<span class="commentTitle">P_loss_coord :</span> 1:181452652-181777219</span></div>	</td>
	<td data-search="3"><div class="tooltip">3<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 3<span class="commentTitle">P_snvindel_nb :</span> 3<br><span class="commentTitle">P_snvindel_phen :</span> DEVELOPMENTAL_AND_EPILEPTIC_ENCEPHALOPATHY_69; Inborn_genetic_diseases</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="39" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="LAMC1"><div class="tooltip">LAMC1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LAMC1<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 35.36<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.888823985988845<br><span class="commentTitle">ExAC_delZ :</span> 0.260807980085751<br><span class="commentTitle">ExAC_dupZ :</span> -869.916.233<br><span class="commentTitle">ExAC_synZ :</span> 0.276289769044398<br><span class="commentTitle">ExAC_misZ :</span> 0.89840821400242<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_002293<br><span class="commentTitle">Exon_count :</span> 28<br><span class="commentTitle">Overlapped_tx_length :</span> 122173<br><span class="commentTitle">Overlapped_CDS_length :</span> 4830<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_174417588_191888383_DEL_1" >
	<td data-order="40" data-search="1_174417588_191888383_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:174417588-191888383&hgt.out1=submit&highlight=hg19.chr1:174417588-191888383#aaedff" target="_blank" rel="noopener noreferrer"">1_174417588_191888383_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_174417588_191888383_DEL_1<br><span class="commentTitle">SV_length :</span> -17470795</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="DHX9"><div class="tooltip">DHX9<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> DHX9<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 10.4<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.986924053965172<br><span class="commentTitle">ExAC_delZ :</span> 0.843805888115509<br><span class="commentTitle">ExAC_dupZ :</span> 0.843104179160815<br><span class="commentTitle">ExAC_synZ :</span> -1.191.392.815<br><span class="commentTitle">ExAC_misZ :</span> -1.964.274.573<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001357<br><span class="commentTitle">Exon_count :</span> 28<br><span class="commentTitle">Overlapped_tx_length :</span> 48636<br><span class="commentTitle">Overlapped_CDS_length :</span> 3813<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr id="1_4629201_15296111_DEL_1" class="full" style="background-color:#F7D8DD" >
	<td data-order="41" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910<br><span class="commentTitle">Gene_count :</span> 158</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="full"><div class="tooltip">full<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> full</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="AADACL3; AADACL4; ACOT7; AGTRAP; AJAP1; ANGPTL7; C1orf127; C1orf158; C1orf167; C1orf167-AS1; CA6; CAMTA1; CAMTA1-AS2; CAMTA1-DT; CASZ1; CENPS; CENPS-CORT; CHD5; CLCN6; CLSTN1; CORT; CTNNBIP1; DFFA; DHRS3; DISP3; DNAJC11; DRAXIN; ENO1; ENO1-AS1; ERRFI1; ESPN; EXOSC10; EXOSC10-AS1; FBXO2; FBXO44; FBXO6; GPR153; GPR157; H6PD; HES2; HES3; HNRNPCL1; HNRNPCL2; HNRNPCL3; HNRNPCL4; ICMT; KAZN; KAZN-AS1; KCNAB2; KIAA2013; KIF1B; KLHL21; LINC00337; LINC01646; LINC01647; LINC01672; LINC01714; LINC01784; LINC02606; LNCTAM34A; LRRC38; LZIC; MAD2L2; MASP2; MFN2; MIIP; MIR34A; MIR34AHG; MIR4252; MIR4632; MIR4689; MIR5697; MIR6728; MIR6729; MIR6730; MIR7846; MTHFR; MTOR; MTOR-AS1; NMNAT1; NOL9; NPHP4; NPPA; NPPA-AS1; NPPB; PARK7; PDPN; PER3; PEX14; PGD; PHF13; PIK3CD; PIK3CD-AS1; PIK3CD-AS2; PLEKHG5; PLOD1; PRAMEF1; PRAMEF10; PRAMEF11; PRAMEF12; PRAMEF13; PRAMEF14; PRAMEF15; PRAMEF17; PRAMEF18; PRAMEF19; PRAMEF2; PRAMEF20; PRAMEF22; PRAMEF25; PRAMEF26; PRAMEF27; PRAMEF33; PRAMEF34P; PRAMEF36P; PRAMEF4; PRAMEF5; PRAMEF6; PRAMEF7; PRAMEF8; PRAMEF9; PRDM2; RBP7; RERE; RERE-AS1; RNF207; RNF207-AS1; RNU5E-1; RNU6-1; RNU6-2; RNU6-7; RNU6-8; RNU6-9; RPL22; SLC25A33; SLC2A5; SLC2A7; SLC45A1; SNORA59A; SNORA59B; SNORD128; SPSB1; SRM; TARDBP; TAS1R1; THAP3; TMEM201; TMEM274P; TNFRSF1B; TNFRSF25; TNFRSF8; TNFRSF9; UBE4B; UBIAD1; UTS2; VAMP3; VPS13D; ZBTB48"><div class="tooltip">AADACL3; AADACL4; ACOT7; AGTRAP; AJAP1; ANGPT[...]<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> AADACL3; AADACL4; ACOT7; AGTRAP; AJAP1; ANGPTL7; C1orf127; C1orf158; C1orf167; C1orf167-AS1; CA6; CAMTA1; CAMTA1-AS2; CAMTA1-DT; CASZ1; CENPS; CENPS-CORT; CHD5; CLCN6; CLSTN1; CORT; CTNNBIP1; DFFA; DHRS3; DISP3; DNAJC11; DRAXIN; ENO1; ENO1-AS1; ERRFI1; ESPN; EXOSC10; EXOSC10-AS1; FBXO2; FBXO44; FBXO6; GPR153; GPR157; H6PD; HES2; HES3; HNRNPCL1; HNRNPCL2; HNRNPCL3; HNRNPCL4; ICMT; KAZN; KAZN-AS1; KCNAB2; KIAA2013; KIF1B; KLHL21; LINC00337; LINC01646; LINC01647; LINC01672; LINC01714; LINC01784; LINC02606; LNCTAM34A; LRRC38; LZIC; MAD2L2; MASP2; MFN2; MIIP; MIR34A; MIR34AHG; MIR4252; MIR4632; MIR4689; MIR5697; MIR6728; MIR6729; MIR6730; MIR7846; MTHFR; MTOR; MTOR-AS1; NMNAT1; NOL9; NPHP4; NPPA; NPPA-AS1; NPPB; PARK7; PDPN; PER3; PEX14; PGD; PHF13; PIK3CD; PIK3CD-AS1; PIK3CD-AS2; PLEKHG5; PLOD1; PRAMEF1; PRAMEF10; PRAMEF11; PRAMEF12; PRAMEF13; PRAMEF14; PRAMEF15; PRAMEF17; PRAMEF18; PRAMEF19; PRAMEF2; PRAMEF20; PRAMEF22; PRAMEF25; PRAMEF26; PRAMEF27; PRAMEF33; PRAMEF34P; PRAMEF36P; PRAMEF4; PRAMEF5; PRAMEF6; PRAMEF7; PRAMEF8; PRAMEF9; PRDM2; RBP7; RERE; RERE-AS1; RNF207; RNF207-AS1; RNU5E-1; RNU6-1; RNU6-2; RNU6-7; RNU6-8; RNU6-9; RPL22; SLC25A33; SLC2A5; SLC2A7; SLC45A1; SNORA59A; SNORA59B; SNORD128; SPSB1; SRM; TARDBP; TAS1R1; THAP3; TMEM201; TMEM274P; TNFRSF1B; TNFRSF25; TNFRSF8; TNFRSF9; UBE4B; UBIAD1; UTS2; VAMP3; VPS13D; ZBTB48<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> .<br><span class="commentTitle">Overlapped_tx_length :</span> .<br><span class="commentTitle">Overlapped_CDS_length :</span> .<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="AGMAT; C1orf174; C1orf195; CASP9; CEP104; EFHD2; FHAD1; PLEKHM2; SCARNA11; SCARNA16; SNORA70; SNORA77; SPEN (HI=1; TS=0); TMEM51; TMEM51-AS1; TP73-AS1"><div class="tooltip">AGMAT; C1orf174; C1orf195; CASP9; CEP104; EFH[...]<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> AGMAT; C1orf174; C1orf195; CASP9; CEP104; EFHD2; FHAD1; PLEKHM2; SCARNA11; SCARNA16; SNORA70; SNORA77; SPEN (HI=1; TS=0); TMEM51; TMEM51-AS1; TP73-AS1</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:10003532-10045556; 1:10290821-10368655; 1:10535006-10690815; 1:11072710-11085549; 1:11104854-11107296; 1:11166591-11322608; 1:11333254-11359628; 1:11734536-11751678; 1:11845786-11863734; 1:11905765-11907840; 1:11994773-12035594; 1:12040500-12073565; 1:12290086-12572098; 1:5922870-6052485; 1:6484835-6521004; 1:6527181-6580152; 1:6534120-6534211; 1:6845384-7829766; 1:7682000-7685000; 1:7764795-7829766; 1:7844488-7905237; 1:8021770-8045565; 1:8378173-8404225; 1:8412463-8877699; 1:8921058-8938745; 1:9299905-9331394; 1:9751524-9788980"><div class="tooltip">1:10003532-10045556; 1:10290821-10368655; 1:1[...]<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:10003532-10045556; 1:10290821-10368655; 1:10535006-10690815; 1:11072710-11085549; 1:11104854-11107296; 1:11166591-11322608; 1:11333254-11359628; 1:11734536-11751678; 1:11845786-11863734; 1:11905765-11907840; 1:11994773-12035594; 1:12040500-12073565; 1:12290086-12572098; 1:5922870-6052485; 1:6484835-6521004; 1:6527181-6580152; 1:6534120-6534211; 1:6845384-7829766; 1:7682000-7685000; 1:7764795-7829766; 1:7844488-7905237; 1:8021770-8045565; 1:8378173-8404225; 1:8412463-8877699; 1:8921058-8938745; 1:9299905-9331394; 1:9751524-9788980<span class="commentTitle">P_loss_coord :</span> 1:10003532-10045556; 1:10290821-10368655; 1:10535006-10690815; 1:11072710-11085549; 1:11104854-11107296; 1:11166591-11322608; 1:11333254-11359628; 1:11734536-11751678; 1:11845786-11863734; 1:11905765-11907840; 1:11994773-12035594; 1:12040500-12073565; 1:12290086-12572098; 1:5922870-6052485; 1:6484835-6521004; 1:6527181-6580152; 1:6534120-6534211; 1:6845384-7829766; 1:7682000-7685000; 1:7764795-7829766; 1:7844488-7905237; 1:8021770-8045565; 1:8378173-8404225; 1:8412463-8877699; 1:8921058-8938745; 1:9299905-9331394; 1:9751524-9788980<br><span class="commentTitle">P_loss_source :</span> CLN:590571; CLN:628239; HI3:CAMTA1; morbid:CAMTA1; morbid:ENO1; morbid:ESPN; morbid:H6PD; morbid:KIF1B; morbid:MAD2L2; morbid:MASP2; morbid:MFN2; morbid:MTHFR; morbid:MTOR; morbid:NMNAT1; morbid:NPHP4; morbid:NPPA; morbid:PARK7; morbid:PER3; morbid:PEX14; morbid:PIK3CD; morbid:PLEKHG5; morbid:PLOD1; morbid:RERE; morbid:SLC45A1; morbid:TARDBP; morbid:UBIAD1; morbid:VPS13D<br><span class="commentTitle">P_loss_phen :</span> "?Advanced sleep phase syndrome, familial, 3, 616882 (3); ?Charcot-Marie-Tooth disease, type 2A1, 118210 (3); ?Fanconi anemia, complementation group V, 617243 (3); ?Usher syndrome, type 1M, 618632 (3); Amyotrophic lateral sclerosis 10, with or without FTD, 612069 (3); Atrial fibrillation, familial, 6, 612201 (3); Atrial standstill 2, 615745 (3); Cerebellar ataxia, nonprogressive, with mental retardation, 614756 (3); Cerebellar_ataxia,_nonprogressive,_with_mental_retardation; Charcot-Marie-Tooth disease, axonal, type 2A2A, 609260 (3); Charcot-Marie-Tooth disease, axonal, type 2A2B, 617087 (3); Charcot-Marie-Tooth disease, recessive intermediate C, 615376 (3); Corneal dystrophy, Schnyder type, 121800 (3); Cortisone reductase deficiency 1, 604931 (3); Deafness, autosomal recessive 36, 609006 (3); Deafness, neurosensory, without vestibular involvement, autosomal dominant, 609006 (3); Distal_spinal_muscular_atrophy,_autosomal_recessive_4|Charcot-Marie-Tooth_disease,_recessive_intermediate_c; Ehlers-Danlos syndrome, kyphoscoliotic type, 1, 225400 (3); Enolase deficiency (1); Focal cortical dysplasia, type II, somatic, 607341 (3); Frontotemporal lobar degeneration, TARDBP-related, 612069 (3); Hereditary motor and sensory neuropathy VIA, 601152 (3); Homocystinuria due to MTHFR deficiency, 236250 (3); Immunodeficiency 14, 615513 (3); Intellectual developmental disorder with neuropsychiatric features, 617532 (3); Leber congenital amaurosis 9, 608553 (3); MASP2 deficiency, 613791 (3); Nephronophthisis 4, 606966 (3); Neural tube defects, susceptibility to, 601634 (3); Neuroblastoma, susceptibility to, 1, 256700 (3); Neurodevelopmental disorder with or without anomalies of the brain, eye, or heart, 616975 (3); Parkinson disease 7, autosomal recessive early-onset, 606324 (3); Peroxisome biogenesis disorder 13A (Zellweger), 614887 (3); Pheochromocytoma, 171300 (3); Schizophrenia, susceptibility to, 181500 (3); Senior-Loken syndrome 4, 606996 (3); Smith-Kingsmore syndrome, 616638 (3); Spinal muscular atrophy, distal, autosomal recessive, 4, 611067 (3); Spinocerebellar ataxia, autosomal recessive 4, 607317 (3); Thromboembolism, susceptibility to, 188050 (3); Vascular disease, susceptibility to (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:10003532-10045556; 1:10290821-10368655; 1:10535006-10690815; 1:11072710-11085549; 1:11104854-11107296; 1:11166591-11322608; 1:11333254-11359628; 1:11734536-11751678; 1:11845786-11863734; 1:11905765-11907840; 1:11994773-12035594; 1:12040500-12073565; 1:12290086-12572098; 1:5922870-6052485; 1:6484835-6521004; 1:6527181-6580152; 1:6534120-6534211; 1:6845384-7829766; 1:7682000-7685000; 1:7764795-7829766; 1:7844488-7905237; 1:8021770-8045565; 1:8378173-8404225; 1:8412463-8877699; 1:8921058-8938745; 1:9299905-9331394; 1:9751524-9788980"><div class="tooltip">1:10003532-10045556; 1:10290821-10368655; 1:1[...]<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:10003532-10045556; 1:10290821-10368655; 1:10535006-10690815; 1:11072710-11085549; 1:11104854-11107296; 1:11166591-11322608; 1:11333254-11359628; 1:11734536-11751678; 1:11845786-11863734; 1:11905765-11907840; 1:11994773-12035594; 1:12040500-12073565; 1:12290086-12572098; 1:5922870-6052485; 1:6484835-6521004; 1:6527181-6580152; 1:6534120-6534211; 1:6845384-7829766; 1:7682000-7685000; 1:7764795-7829766; 1:7844488-7905237; 1:8021770-8045565; 1:8378173-8404225; 1:8412463-8877699; 1:8921058-8938745; 1:9299905-9331394; 1:9751524-9788980<span class="commentTitle">P_loss_coord :</span> 1:10003532-10045556; 1:10290821-10368655; 1:10535006-10690815; 1:11072710-11085549; 1:11104854-11107296; 1:11166591-11322608; 1:11333254-11359628; 1:11734536-11751678; 1:11845786-11863734; 1:11905765-11907840; 1:11994773-12035594; 1:12040500-12073565; 1:12290086-12572098; 1:5922870-6052485; 1:6484835-6521004; 1:6527181-6580152; 1:6534120-6534211; 1:6845384-7829766; 1:7682000-7685000; 1:7764795-7829766; 1:7844488-7905237; 1:8021770-8045565; 1:8378173-8404225; 1:8412463-8877699; 1:8921058-8938745; 1:9299905-9331394; 1:9751524-9788980</span></div>	</td>
	<td data-search="190"><div class="tooltip">190<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 190<span class="commentTitle">P_snvindel_nb :</span> 190<br><span class="commentTitle">P_snvindel_phen :</span> "Abnormal_blistering_of_the_skin; Abnormal_lower_motor_neuron_morphology; Abnormality_of_dental_enamel; Abnormality_of_the_dentition; Acquired_porencephaly; Advanced_sleep_phase_syndrome,_familial,_3; Alopecia_of_scalp; Amyotrophic_lateral_sclerosis-parkinsonism-dementia_complex; Amyotrophic_lateral_sclerosis_type_10; Autosomal_recessive_cerebellar_ataxia-saccadic_intrusion_syndrome; Bilateral_cryptorchidism; Breast_neoplasm; CEBALID_syndrome; CHARGE_association; Cardiovascular_phenotype; Cerebellar_ataxia; Cerebellar_ataxia,_nonprogressive,_with_mental_retardation; Charcot-Marie-Tooth_disease; Charcot-Marie-Tooth_disease,_axonal,_autosomal_recessive,_type_2A2B; Charcot-Marie-Tooth_disease,_recessive_intermediate_c; Charcot-Marie-Tooth_disease,_type_2; Charcot-Marie-Tooth_disease,_type_2A2A; Charcot-Marie-Tooth_disease_type_4; Chronic_kidney_disease; Cone-rod_dystrophy; Congenital_omphalocele; Deafness,_autosomal_recessive_36,_with_or_without_vestibular_involvement; Decreased_body_weight; Depressed_nasal_bridge; Developmental_regression; Diarrhea; Distal_lower_limb_amyotrophy; Distal_muscle_weakness; Distal_spinal_muscular_atrophy,_autosomal_recessive_4; Dolichocephaly; EMG_abnormality; Ehlers-Danlos_syndrome,_hydroxylysine-deficient; Failure_to_thrive; Feeding_difficulties; Focal_cortical_dysplasia_type_II; Gastrointestinal_dysmotility; Generalized_hypotonia; Generalized_neonatal_hypotonia; Genetic_motor_neuron_disease; Glioblastoma; Global_developmental_delay; Hematuria; Hereditary_motor_and_sensory_neuropathy_with_optic_atrophy; High_palate; Homocystinuria_due_to_MTHFR_deficiency; Homocystinuria_due_to_methylene_tetrahydrofolate_reductase_deficiency; Hydrocephalus; Hyperpigmentation_of_the_skin; Hyporeflexia; Immunodeficiency_14; Inborn_genetic_diseases; Inherited_Immunodeficiency_Diseases; Intellectual_disability; Intellectual_disability,_severe; Joint_hypermobility; Juvenile_amyotrophic_lateral_sclerosis; Leber_congenital_amaurosis; Leber_congenital_amaurosis_9; MFN2-Related_Disorders; MTOR-related_megalencephaly_and_pigmentary_mosaicism_in_skin; Macrocephaly_at_birth; Microcephaly; Motor_neuron_disease; Nail_dystrophy; Narrow_chest; Neonatal_hypotonia; Nephronophthisis; Nephronophthisis_4; Neural_tube_defects,_folate-sensitive; Neurodevelopmental_disorder_with_or_without_anomalies_of_the_brain,_eye,_or_heart; Nystagmus; Parkinson_disease_7; Peripheral_axonal_neuropathy; Proteinuria; Rare_genetic_intellectual_disability; Renal_cell_carcinoma,_papillary,_1; Retinal_dystrophy; Scarring; Scarring_alopecia_of_scalp; Scrotal_hypoplasia; Senior-Loken_syndrome_4; Severe_global_developmental_delay; Severe_visual_impairment; Short_chin; Short_stature; Smith-Kingsmore_syndrome; TARDBP-related_frontotemporal_dementia; Thoracolumbar_scoliosis; Transitional_cell_carcinoma_of_the_bladder; USHER_SYNDROME,_TYPE_1M; Umbilical_hernia; Usher_syndrome_type_1; not_specified"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .<br><span class="commentTitle">ENCODE_blacklist_left :</span> .<br><span class="commentTitle">SegDup_left :</span> .<br><span class="commentTitle">Repeat_coord_left :</span> .<br><span class="commentTitle">Repeat_type_left :</span> .<br><span class="commentTitle">Gap_left :</span> .<br><span class="commentTitle">GC_content_left :</span> 0.445</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .<br><span class="commentTitle">ENCODE_blacklist_right :</span> .<br><span class="commentTitle">SegDup_right :</span> .<br><span class="commentTitle">Repeat_coord_right :</span> .<br><span class="commentTitle">Repeat_type_right :</span> .<br><span class="commentTitle">Gap_right :</span> .<br><span class="commentTitle">GC_content_right :</span> 0.405</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="42" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR4689"><div class="tooltip">MIR4689<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR4689<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_039838<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 70<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="43" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RNU5E-1"><div class="tooltip">RNU5E-1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNU5E-1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_002754<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 119<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="44" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="C1orf167-AS1"><div class="tooltip">C1orf167-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C1orf167-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_126000<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 2543<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="45" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="CAMTA1-AS2"><div class="tooltip">CAMTA1-AS2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CAMTA1-AS2<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_146199<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 7267<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="46" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RNU6-9"><div class="tooltip">RNU6-9<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNU6-9<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_104080<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 105<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="47" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01672"><div class="tooltip">LINC01672<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01672<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_104620<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 5268<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="48" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR34A"><div class="tooltip">MIR34A<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR34A<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_029610<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 110<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="49" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR5697"><div class="tooltip">MIR5697<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR5697<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_049882<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 78<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="50" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR6729"><div class="tooltip">MIR6729<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR6729<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_106787<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 65<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="51" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF19"><div class="tooltip">PRAMEF19<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF19<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "5,97E+02"<br><span class="commentTitle">ExAC_pLI :</span> "5,83E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 95.67<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -5.597.728<br><span class="commentTitle">ExAC_misZ :</span> 302.015.894<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001099790<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 3558<br><span class="commentTitle">Overlapped_CDS_length :</span> 1440<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="52" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RNU6-1"><div class="tooltip">RNU6-1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNU6-1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_004394<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 105<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="53" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01646"><div class="tooltip">LINC01646<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01646<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_147025<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 12218<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="54" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="CAMTA1-DT"><div class="tooltip">CAMTA1-DT<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CAMTA1-DT<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_149049<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 1089<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="55" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="NPPA-AS1"><div class="tooltip">NPPA-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NPPA-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_037806<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 7298<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="56" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC00337"><div class="tooltip">LINC00337<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC00337<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_103534<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 3203<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="57" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MTOR-AS1"><div class="tooltip">MTOR-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MTOR-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_046600<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5641<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="1"><div class="tooltip">1<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 1<span class="commentTitle">P_snvindel_nb :</span> 1<br><span class="commentTitle">P_snvindel_phen :</span> Smith-Kingsmore_syndrome</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="58" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF13"><div class="tooltip">PRAMEF13<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF13<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "2,92E+03"<br><span class="commentTitle">ExAC_pLI :</span> "5,29E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.73<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -1.390.725.541<br><span class="commentTitle">ExAC_misZ :</span> -941.318.458<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001291380<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5243<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="59" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR6728"><div class="tooltip">MIR6728<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR6728<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_106786<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 89<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="60" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RNU6-2"><div class="tooltip">RNU6-2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNU6-2<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_125730<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 105<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="61" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="TMEM274P"><div class="tooltip">TMEM274P<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TMEM274P<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_158969<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 10216<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="62" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF4"><div class="tooltip">PRAMEF4<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF4<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "1,02E-07"<br><span class="commentTitle">ExAC_pLI :</span> "7,46E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.8<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -611.973.269<br><span class="commentTitle">ExAC_misZ :</span> 674.753.911<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001009611<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 6993<br><span class="commentTitle">Overlapped_CDS_length :</span> 1437<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:468; DDD:470; dg[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662<br><span class="commentTitle">B_loss_coord :</span> 1:12841928-12971833; 1:12854390-13146372; 1:12877893-12948772; 1:12893536-12948772; 1:12898653-13013316; 1:12901912-12959442; 1:12907342-13183072; 1:12935657-13049137</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:468; DDD:470; dg[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662<br><span class="commentTitle">B_loss_coord :</span> 1:12841928-12971833; 1:12854390-13146372; 1:12877893-12948772; 1:12893536-12948772; 1:12898653-13013316; 1:12901912-12959442; 1:12907342-13183072; 1:12935657-13049137</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="63" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="DISP3"><div class="tooltip">DISP3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> DISP3<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_020780<br><span class="commentTitle">Exon_count :</span> 21<br><span class="commentTitle">Overlapped_tx_length :</span> 58397<br><span class="commentTitle">Overlapped_CDS_length :</span> 4179<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="64" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF14"><div class="tooltip">PRAMEF14<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF14<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "2,90E+03"<br><span class="commentTitle">ExAC_pLI :</span> "5,60E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 97.07<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -1.129.127.179<br><span class="commentTitle">ExAC_misZ :</span> 1.356.061.648<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001024661<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5243<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="65" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF22"><div class="tooltip">PRAMEF22<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF22<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "4,17E+01"<br><span class="commentTitle">ExAC_pLI :</span> "6,17E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.13<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 1.507.226.085<br><span class="commentTitle">ExAC_misZ :</span> -1.597.930.869<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001100631<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 148696<br><span class="commentTitle">Overlapped_CDS_length :</span> 1440<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="66" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PIK3CD-AS1"><div class="tooltip">PIK3CD-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PIK3CD-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.96<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_027045<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 1977<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="67" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR7846"><div class="tooltip">MIR7846<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR7846<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_107000<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 96<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="68" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RNU6-7"><div class="tooltip">RNU6-7<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNU6-7<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_104084<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 105<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="69" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF18"><div class="tooltip">PRAMEF18<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF18<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "3,04E+03"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.58<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001099850<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 370210<br><span class="commentTitle">Overlapped_CDS_length :</span> 1440<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="70" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LNCTAM34A"><div class="tooltip">LNCTAM34A<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LNCTAM34A<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_132738<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 9882<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="71" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RNU6-8"><div class="tooltip">RNU6-8<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNU6-8<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_104088<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 105<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="72" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF9"><div class="tooltip">PRAMEF9<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF9<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.55<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001010890<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 3849<br><span class="commentTitle">Overlapped_CDS_length :</span> 1436<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> yes<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137; 1:12984999-13029500</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137; 1:12984999-13029500</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="73" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="HNRNPCL4"><div class="tooltip">HNRNPCL4<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HNRNPCL4<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001302551<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 1712<br><span class="commentTitle">Overlapped_CDS_length :</span> 882<br><span class="commentTitle">Overlapped_CDS_percent :</span> 84<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000; 1:13178772-13187474; 1:13178772-13187516</span></div>	</td>
	<td data-search="1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000; 1:13178772-13187474; 1:13178772-13187516</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="74" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF36P"><div class="tooltip">PRAMEF36P<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF36P<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_111945<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5210<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="DDD:510; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000</span></div>	</td>
	<td data-search="DDD:510; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="75" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF34P"><div class="tooltip">PRAMEF34P<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF34P<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_111947<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5210<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="DDD:510; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000</span></div>	</td>
	<td data-search="DDD:510; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> DDD:510; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="76" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="KAZN-AS1"><div class="tooltip">KAZN-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KAZN-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_149058<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 71020<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="77" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF8"><div class="tooltip">PRAMEF8<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF8<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 97.46<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001012276<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 3834<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:456; DDD:467; DDD:470<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="78" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RNF207-AS1"><div class="tooltip">RNF207-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNF207-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_125997<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 941<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="79" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR34AHG"><div class="tooltip">MIR34AHG<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR34AHG<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_132742<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 34328<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="80" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="SNORD128"><div class="tooltip">SNORD128<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SNORD128<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_132752<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 114<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="81" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR6730"><div class="tooltip">MIR6730<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR6730<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_106788<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 67<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="82" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="CORT"><div class="tooltip">CORT<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CORT<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "1,31E+02"<br><span class="commentTitle">ExAC_pLI :</span> "9,84E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 67.02<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.239312662927943<br><span class="commentTitle">ExAC_delZ :</span> 0.374335892713973<br><span class="commentTitle">ExAC_dupZ :</span> 0.0420265823943045<br><span class="commentTitle">ExAC_synZ :</span> 0.0737457356746673<br><span class="commentTitle">ExAC_misZ :</span> -0.784438770076526<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001302<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 1968<br><span class="commentTitle">Overlapped_CDS_length :</span> 318<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="83" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF27"><div class="tooltip">PRAMEF27<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF27<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001300891<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 65439<br><span class="commentTitle">Overlapped_CDS_length :</span> 1437<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="DDD:456; DDD:470"><div class="tooltip">DDD:456; DDD:470<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> DDD:456; DDD:470<span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12907342-13183072</span></div>	</td>
	<td data-search="DDD:456; DDD:470"><div class="tooltip">DDD:456; DDD:470<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12907342-13183072</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="84" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC02606"><div class="tooltip">LINC02606<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC02606<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_104622<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 3903<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="85" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="EXOSC10-AS1"><div class="tooltip">EXOSC10-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> EXOSC10-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_135061<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 2431<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="86" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="HNRNPCL2"><div class="tooltip">HNRNPCL2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HNRNPCL2<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001136561<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 1367<br><span class="commentTitle">Overlapped_CDS_length :</span> 882<br><span class="commentTitle">Overlapped_CDS_percent :</span> 82<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000; 1:13178772-13187474; 1:13178772-13187516; 1:13182563-13196202</span></div>	</td>
	<td data-search="1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000; 1:13178772-13187474; 1:13178772-13187516; 1:13182563-13196202</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="87" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF5"><div class="tooltip">PRAMEF5<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF5<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.59<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001013407<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 9260<br><span class="commentTitle">Overlapped_CDS_length :</span> 1431<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="88" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01714"><div class="tooltip">LINC01714<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01714<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_125998<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 6537<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="89" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01647"><div class="tooltip">LINC01647<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01647<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_104624<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 3824<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="90" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF26"><div class="tooltip">PRAMEF26<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF26<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "3,31E-03"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 95.76<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001306072<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 60309<br><span class="commentTitle">Overlapped_CDS_length :</span> 1437<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="DDD:456; DDD:470"><div class="tooltip">DDD:456; DDD:470<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> DDD:456; DDD:470<span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12907342-13183072</span></div>	</td>
	<td data-search="DDD:456; DDD:470"><div class="tooltip">DDD:456; DDD:470<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12907342-13183072</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="91" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF2"><div class="tooltip">PRAMEF2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF2<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "4,74E-06"<br><span class="commentTitle">ExAC_pLI :</span> "5,68E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.05<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -0.017068265132357<br><span class="commentTitle">ExAC_misZ :</span> -0.970014143276301<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_023014<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 4824<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662; esv3585246; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085"><div class="tooltip">1000g; DDD:446; DDD:456; DDD:467; DDD:468; DD[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:446; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662; esv3585246; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662; esv3585246; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12854390-13146372; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772; 1:12893536-12948772; 1:12896424-12930275; 1:12896438-12928826; 1:12896438-12928826; 1:12898653-13013316; 1:12901912-12959442; 1:12907342-13183072</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662; esv3585246; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085"><div class="tooltip">1000g; DDD:446; DDD:456; DDD:467; DDD:468; DD[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662; esv3585246; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:456; DDD:467; DDD:468; DDD:470; dgv6e214; esv2421662; esv3585246; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12854390-13146372; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772; 1:12893536-12948772; 1:12896424-12930275; 1:12896438-12928826; 1:12896438-12928826; 1:12898653-13013316; 1:12901912-12959442; 1:12907342-13183072</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="92" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="SNORA59B"><div class="tooltip">SNORA59B<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SNORA59B<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_003022<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 152<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="93" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF15"><div class="tooltip">PRAMEF15<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF15<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "3,29E+03"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.84<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001098376<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 7018<br><span class="commentTitle">Overlapped_CDS_length :</span> 1437<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="94" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF17"><div class="tooltip">PRAMEF17<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF17<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "5,32E-04"<br><span class="commentTitle">ExAC_pLI :</span> "7,91E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.23<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 1.268.770.931<br><span class="commentTitle">ExAC_misZ :</span> -1.223.046.835<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001099851<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 3000<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1133"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1133<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1133<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13493999-13510000</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1133"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1133<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1133<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13493999-13510000</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="95" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="ENO1-AS1"><div class="tooltip">ENO1-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ENO1-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_038351<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 1050<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="96" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF20"><div class="tooltip">PRAMEF20<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF20<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "3,02E+01"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.29<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001099852<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 10897<br><span class="commentTitle">Overlapped_CDS_length :</span> 1428<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="dgv16e199; gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">dgv16e199; gnomAD-SV_v2.1_DEL_1_1116<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> dgv16e199; gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> dgv16e199; gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13692845-13774592</span></div>	</td>
	<td data-search="dgv16e199; gnomAD-SV_v2.1_DEL_1_1116"><div class="tooltip">dgv16e199; gnomAD-SV_v2.1_DEL_1_1116<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> dgv16e199; gnomAD-SV_v2.1_DEL_1_1116<span class="commentTitle">B_loss_source :</span> dgv16e199; gnomAD-SV_v2.1_DEL_1_1116<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13692845-13774592</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="97" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="HNRNPCL1"><div class="tooltip">HNRNPCL1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HNRNPCL1<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "3,73E+03"<br><span class="commentTitle">ExAC_pLI :</span> "1,07E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 90.59<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -0.725647943609533<br><span class="commentTitle">ExAC_misZ :</span> 1.165.818.393<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001013631<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 1344<br><span class="commentTitle">Overlapped_CDS_length :</span> 882<br><span class="commentTitle">Overlapped_CDS_percent :</span> 82<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:455; DDD:456; DDD:467; DDD:468; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585246; esv3585247; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085"><div class="tooltip">1000g; DDD:446; DDD:455; DDD:456; DDD:467; DD[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:446; DDD:455; DDD:456; DDD:467; DDD:468; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585246; esv3585247; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:455; DDD:456; DDD:467; DDD:468; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585246; esv3585247; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12853865-12919353; 1:12854390-13146372; 1:12854895-12919194; 1:12854895-12919194; 1:12860749-12913896; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772; 1:12893536-12948772; 1:12896424-12930275; 1:12896438-12928826; 1:12896438-12928826; 1:12898653-13013316; 1:12901370-12921250; 1:12901370-12921250; 1:12901912-12959442</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:455; DDD:456; DDD:467; DDD:468; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585246; esv3585247; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085"><div class="tooltip">1000g; DDD:446; DDD:455; DDD:456; DDD:467; DD[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:455; DDD:456; DDD:467; DDD:468; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585246; esv3585247; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:455; DDD:456; DDD:467; DDD:468; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585246; esv3585247; gnomAD-SV_v2.1_DEL_1_1083; gnomAD-SV_v2.1_DEL_1_1085<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12853865-12919353; 1:12854390-13146372; 1:12854895-12919194; 1:12854895-12919194; 1:12860749-12913896; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772; 1:12893536-12948772; 1:12896424-12930275; 1:12896438-12928826; 1:12896438-12928826; 1:12898653-13013316; 1:12901370-12921250; 1:12901370-12921250; 1:12901912-12959442</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="98" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF7"><div class="tooltip">PRAMEF7<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF7<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "3,31E+03"<br><span class="commentTitle">ExAC_pLI :</span> "1,77E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.25<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 692.411.350<br><span class="commentTitle">ExAC_misZ :</span> 52.524.000<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001012277<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 3834<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:456; DDD:467; DDD:470<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="99" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="HNRNPCL3"><div class="tooltip">HNRNPCL3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HNRNPCL3<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001382358<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 2102<br><span class="commentTitle">Overlapped_CDS_length :</span> 882<br><span class="commentTitle">Overlapped_CDS_percent :</span> 82<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000; 1:13178772-13187474; 1:13178772-13187516</span></div>	</td>
	<td data-search="1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108"><div class="tooltip">1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<span class="commentTitle">B_loss_source :</span> 1000g; DDD:510; DDD:522; gnomAD-SV_v2.1_DEL_1_1108<br><span class="commentTitle">B_loss_coord :</span> 1:13132173-13215470; 1:13145449-13216000; 1:13178772-13187474; 1:13178772-13187516</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="100" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01784"><div class="tooltip">LINC01784<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01784<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_146629<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 538<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1079; gnomAD-SV_v2.1_DEL_1_1083"><div class="tooltip">1000g; DDD:446; DDD:450; DDD:455; DDD:456; dg[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1079; gnomAD-SV_v2.1_DEL_1_1083<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1079; gnomAD-SV_v2.1_DEL_1_1083<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12848806-12903525; 1:12851499-12890000; 1:12853865-12919353; 1:12854390-13146372; 1:12854895-12919194; 1:12854895-12919194; 1:12856327-12893193; 1:12856327-12893193; 1:12860749-12913896; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1079; gnomAD-SV_v2.1_DEL_1_1083"><div class="tooltip">1000g; DDD:446; DDD:450; DDD:455; DDD:456; dg[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1079; gnomAD-SV_v2.1_DEL_1_1083<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1079; gnomAD-SV_v2.1_DEL_1_1083<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12848806-12903525; 1:12851499-12890000; 1:12853865-12919353; 1:12854390-13146372; 1:12854895-12919194; 1:12854895-12919194; 1:12856327-12893193; 1:12856327-12893193; 1:12860749-12913896; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="101" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF25"><div class="tooltip">PRAMEF25<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF25<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001310134<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 60463<br><span class="commentTitle">Overlapped_CDS_length :</span> 1437<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="DDD:456; DDD:470"><div class="tooltip">DDD:456; DDD:470<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> DDD:456; DDD:470<span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12907342-13183072</span></div>	</td>
	<td data-search="DDD:456; DDD:470"><div class="tooltip">DDD:456; DDD:470<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<span class="commentTitle">B_loss_source :</span> DDD:456; DDD:470<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12907342-13183072</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="102" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="RERE-AS1"><div class="tooltip">RERE-AS1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RERE-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_125999<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 10372<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="103" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR4252"><div class="tooltip">MIR4252<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR4252<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_036218<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 63<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="DDD:268"><div class="tooltip">DDD:268<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> DDD:268<span class="commentTitle">B_loss_source :</span> DDD:268<br><span class="commentTitle">B_loss_coord :</span> 1:6481741-6515522</span></div>	</td>
	<td data-search="DDD:268"><div class="tooltip">DDD:268<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> DDD:268<span class="commentTitle">B_loss_source :</span> DDD:268<br><span class="commentTitle">B_loss_coord :</span> 1:6481741-6515522</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="104" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="SNORA59A"><div class="tooltip">SNORA59A<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SNORA59A<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_003025<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 152<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="105" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="CENPS"><div class="tooltip">CENPS<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CENPS<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_199294<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 12376<br><span class="commentTitle">Overlapped_CDS_length :</span> 417<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="106" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PIK3CD-AS2"><div class="tooltip">PIK3CD-AS2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PIK3CD-AS2<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_126366<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 15149<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="107" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PRAMEF33"><div class="tooltip">PRAMEF33<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF33<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001291381<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5368<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700; 1:13384749-13419500</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700; 1:13384749-13419500</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="108" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF6"><div class="tooltip">PRAMEF6<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF6<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "2,61E-07"<br><span class="commentTitle">ExAC_pLI :</span> "6,91E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.38<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -1.034.975.097<br><span class="commentTitle">ExAC_misZ :</span> 66.330.567<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001010889<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 9105<br><span class="commentTitle">Overlapped_CDS_length :</span> 1431<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137; 1:12984999-13029500</span></div>	</td>
	<td data-search="1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093"><div class="tooltip">1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<span class="commentTitle">B_loss_source :</span> 1000g; DDD:456; DDD:467; DDD:470; gnomAD-SV_v2.1_DEL_1_1093<br><span class="commentTitle">B_loss_coord :</span> 1:12854390-13146372; 1:12898653-13013316; 1:12907342-13183072; 1:12935657-13049137; 1:12984999-13029500</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="109" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="PRAMEF1"><div class="tooltip">PRAMEF1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF1<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "3,22E-07"<br><span class="commentTitle">ExAC_pLI :</span> "9,81E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.53<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 898.473.014<br><span class="commentTitle">ExAC_misZ :</span> -303.029.453<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_023013<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5232<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_DEL_1_1079"><div class="tooltip">DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_DEL_1_1079<span class="commentTitle">B_loss_source :</span> DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_DEL_1_1079<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12848806-12903525; 1:12851499-12890000</span></div>	</td>
	<td data-search="DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_DEL_1_1079"><div class="tooltip">DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_DEL_1_1079<span class="commentTitle">B_loss_source :</span> DDD:446; DDD:450; esv2421662; gnomAD-SV_v2.1_DEL_1_1079<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12848806-12903525; 1:12851499-12890000</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="110" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="CENPS-CORT"><div class="tooltip">CENPS-CORT<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CENPS-CORT<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_198544<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 21721<br><span class="commentTitle">Overlapped_CDS_length :</span> 495<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="111" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR4632"><div class="tooltip">MIR4632<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR4632<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_039775<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 61<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="112" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="NPPB"><div class="tooltip">NPPB<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NPPB<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> "9,88E-02"<br><span class="commentTitle">ExAC_pLI :</span> "3,80E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 89.09<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.51772512803037<br><span class="commentTitle">ExAC_delZ :</span> 0.361673479958905<br><span class="commentTitle">ExAC_dupZ :</span> 0.378290749278187<br><span class="commentTitle">ExAC_synZ :</span> 0.38618690031033<br><span class="commentTitle">ExAC_misZ :</span> -0.058586851825831<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_002521<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 1482<br><span class="commentTitle">Overlapped_CDS_length :</span> 405<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="113" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="TMEM201"><div class="tooltip">TMEM201<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TMEM201<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "9,90E+03"<br><span class="commentTitle">ExAC_pLI :</span> "8,42E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 58.8<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.202142806692057<br><span class="commentTitle">ExAC_delZ :</span> -0.217989315893959<br><span class="commentTitle">ExAC_dupZ :</span> 0.368030477490982<br><span class="commentTitle">ExAC_synZ :</span> 0.102604713595419<br><span class="commentTitle">ExAC_misZ :</span> 0.849633741989809<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001130924<br><span class="commentTitle">Exon_count :</span> 11<br><span class="commentTitle">Overlapped_tx_length :</span> 25967<br><span class="commentTitle">Overlapped_CDS_length :</span> 2001<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="114" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="PGD"><div class="tooltip">PGD<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PGD<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "9,88E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,69E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 34.45<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 1.399.131.390<br><span class="commentTitle">ExAC_delZ :</span> -1.845.989.135<br><span class="commentTitle">ExAC_dupZ :</span> 0.884951918577283<br><span class="commentTitle">ExAC_synZ :</span> -0.295694546241906<br><span class="commentTitle">ExAC_misZ :</span> 412.256.838<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_002631<br><span class="commentTitle">Exon_count :</span> 13<br><span class="commentTitle">Overlapped_tx_length :</span> 21448<br><span class="commentTitle">Overlapped_CDS_length :</span> 1452<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="115" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background: linear-gradient(-45deg, white 50%, #FF3300 50% )" data-search="KAZN"><div class="tooltip">KAZN<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KAZN<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "5,97E+03"<br><span class="commentTitle">ExAC_pLI :</span> "1,45E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 15.65<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.222130110416872<br><span class="commentTitle">ExAC_delZ :</span> 0.596399052261409<br><span class="commentTitle">ExAC_dupZ :</span> -0.567361044244512<br><span class="commentTitle">ExAC_synZ :</span> 0.298062600105162<br><span class="commentTitle">ExAC_misZ :</span> 203.410.405<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-intron2"><div class="tooltip">txStart-intron2<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-intron2<br><span class="commentTitle">Location2 :</span> 5'UTR-CDS<br><span class="commentTitle">Tx :</span> NM_201628<br><span class="commentTitle">Exon_count :</span> 15<br><span class="commentTitle">Overlapped_tx_length :</span> 370931<br><span class="commentTitle">Overlapped_CDS_length :</span> 418<br><span class="commentTitle">Overlapped_CDS_percent :</span> 17<br><span class="commentTitle">Frameshift :</span> yes<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> 5'<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="116" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="MFN2"><div class="tooltip">MFN2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MFN2<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "9,94E+03"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> 0<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 12.08<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.0420199016413099<br><span class="commentTitle">ExAC_delZ :</span> 0.545892707670917<br><span class="commentTitle">ExAC_dupZ :</span> -0.381389545256497<br><span class="commentTitle">ExAC_synZ :</span> -0.472268551190321<br><span class="commentTitle">ExAC_misZ :</span> -147.310.488<br><span class="commentTitle">DDD_status :</span> both RD and IF<br><span class="commentTitle">DDD_mode :</span> monoallelic<br><span class="commentTitle">DDD_consequence :</span> all missense/ in frame<br><span class="commentTitle">DDD_disease :</span> MFN2-related developmental disorder<br><span class="commentTitle">DDD_pmid :</span> 33057194</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001127660<br><span class="commentTitle">Exon_count :</span> 18<br><span class="commentTitle">Overlapped_tx_length :</span> 33335<br><span class="commentTitle">Overlapped_CDS_length :</span> 2274<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Hereditary motor and sensory neuropathy VIA, 601152 (3)/  Charcot-Marie-Tooth disease, axonal, type 2A2B, 617087 (3)/  Charcot-Marie-Tooth disease, axonal, type 2A2A, 609260 (3)""><div class="tooltip">"Hereditary motor and sensory neuropathy VIA,[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Hereditary motor and sensory neuropathy VIA, 601152 (3)/  Charcot-Marie-Tooth disease, axonal, type 2A2B, 617087 (3)/  Charcot-Marie-Tooth disease, axonal, type 2A2A, 609260 (3)"<br><span class="commentTitle">OMIM_ID :</span> 608507<br><span class="commentTitle">OMIM_inheritance :</span> AD/ AR/ AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:12040500-12073565"><div class="tooltip">1:12040500-12073565<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:12040500-12073565<span class="commentTitle">P_loss_coord :</span> 1:12040500-12073565<br><span class="commentTitle">P_loss_source :</span> morbid:MFN2<br><span class="commentTitle">P_loss_phen :</span> "Charcot-Marie-Tooth disease, axonal, type 2A2A, 609260 (3); Charcot-Marie-Tooth disease, axonal, type 2A2B, 617087 (3); Hereditary motor and sensory neuropathy VIA, 601152 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:12040500-12073565"><div class="tooltip">1:12040500-12073565<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:12040500-12073565<span class="commentTitle">P_loss_coord :</span> 1:12040500-12073565</span></div>	</td>
	<td data-search="55"><div class="tooltip">55<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 55<span class="commentTitle">P_snvindel_nb :</span> 55<br><span class="commentTitle">P_snvindel_phen :</span> "Abnormal_blistering_of_the_skin; Abnormal_lower_motor_neuron_morphology; Abnormality_of_dental_enamel; Abnormality_of_the_dentition; Alopecia_of_scalp; Cerebellar_ataxia; Charcot-Marie-Tooth_disease; Charcot-Marie-Tooth_disease,_axonal,_autosomal_recessive,_type_2A2B; Charcot-Marie-Tooth_disease,_type_2; Charcot-Marie-Tooth_disease,_type_2A2A; Charcot-Marie-Tooth_disease_type_4; Decreased_body_weight; Distal_lower_limb_amyotrophy; Distal_muscle_weakness; EMG_abnormality; Failure_to_thrive; Generalized_hypotonia; Genetic_motor_neuron_disease; Hereditary_motor_and_sensory_neuropathy_with_optic_atrophy; Hyperpigmentation_of_the_skin; Hyporeflexia; Inborn_genetic_diseases; MFN2-Related_Disorders; Microcephaly; Nail_dystrophy; Peripheral_axonal_neuropathy; Scarring; Scarring_alopecia_of_scalp; Short_stature; not_specified"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="117" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="TARDBP"><div class="tooltip">TARDBP<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TARDBP<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "9,85E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,81E+03"<br><span class="commentTitle">HI :</span> 0<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 12.94<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.326435162451961<br><span class="commentTitle">ExAC_delZ :</span> 0.726876409273004<br><span class="commentTitle">ExAC_dupZ :</span> -0.0115968320866207<br><span class="commentTitle">ExAC_synZ :</span> 0.447650848525544<br><span class="commentTitle">ExAC_misZ :</span> 944.529.594<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_007375<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 12839<br><span class="commentTitle">Overlapped_CDS_length :</span> 1245<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Frontotemporal lobar degeneration, TARDBP-related, 612069 (3)/  Amyotrophic lateral sclerosis 10, with or without FTD, 612069 (3)""><div class="tooltip">"Frontotemporal lobar degeneration, TARDBP-re[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Frontotemporal lobar degeneration, TARDBP-related, 612069 (3)/  Amyotrophic lateral sclerosis 10, with or without FTD, 612069 (3)"<br><span class="commentTitle">OMIM_ID :</span> 605078<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:11072710-11085549"><div class="tooltip">1:11072710-11085549<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:11072710-11085549<span class="commentTitle">P_loss_coord :</span> 1:11072710-11085549<br><span class="commentTitle">P_loss_source :</span> morbid:TARDBP<br><span class="commentTitle">P_loss_phen :</span> "Amyotrophic lateral sclerosis 10, with or without FTD, 612069 (3); Frontotemporal lobar degeneration, TARDBP-related, 612069 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:11072710-11085549"><div class="tooltip">1:11072710-11085549<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:11072710-11085549<span class="commentTitle">P_loss_coord :</span> 1:11072710-11085549</span></div>	</td>
	<td data-search="8"><div class="tooltip">8<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 8<span class="commentTitle">P_snvindel_nb :</span> 8<br><span class="commentTitle">P_snvindel_phen :</span> Amyotrophic_lateral_sclerosis_type_10; Motor_neuron_disease; TARDBP-related_frontotemporal_dementia</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="118" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="KCNAB2"><div class="tooltip">KCNAB2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KCNAB2<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "8,00E+03"<br><span class="commentTitle">ExAC_pLI :</span> "8,40E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 21.55<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.214762427385569<br><span class="commentTitle">ExAC_delZ :</span> 0.260806187999501<br><span class="commentTitle">ExAC_dupZ :</span> 0.124406218431882<br><span class="commentTitle">ExAC_synZ :</span> -0.251349144176744<br><span class="commentTitle">ExAC_misZ :</span> -297.738.072<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001199862<br><span class="commentTitle">Exon_count :</span> 16<br><span class="commentTitle">Overlapped_tx_length :</span> 55293<br><span class="commentTitle">Overlapped_CDS_length :</span> 1248<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="119" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="CLSTN1"><div class="tooltip">CLSTN1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CLSTN1<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "9,78E+03"<br><span class="commentTitle">ExAC_pLI :</span> "8,44E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 54.69<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.72539204712497<br><span class="commentTitle">ExAC_delZ :</span> 0.57140167786879<br><span class="commentTitle">ExAC_dupZ :</span> 0.639492130454302<br><span class="commentTitle">ExAC_synZ :</span> -0.718024415069806<br><span class="commentTitle">ExAC_misZ :</span> -371.410.842<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001009566<br><span class="commentTitle">Exon_count :</span> 19<br><span class="commentTitle">Overlapped_tx_length :</span> 95040<br><span class="commentTitle">Overlapped_CDS_length :</span> 2946<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="120" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="VPS13D"><div class="tooltip">VPS13D<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> VPS13D<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "1,07E-02"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 22.87<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 934.619.703<br><span class="commentTitle">ExAC_delZ :</span> 0.086532128555913<br><span class="commentTitle">ExAC_dupZ :</span> 1.371.950.715<br><span class="commentTitle">ExAC_synZ :</span> 0.174044194128562<br><span class="commentTitle">ExAC_misZ :</span> -1.469.140.881<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_015378<br><span class="commentTitle">Exon_count :</span> 70<br><span class="commentTitle">Overlapped_tx_length :</span> 282012<br><span class="commentTitle">Overlapped_CDS_length :</span> 13167<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Spinocerebellar ataxia, AR 4, 607317 (3)""><div class="tooltip">"Spinocerebellar ataxia, AR 4, 607317 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Spinocerebellar ataxia, AR 4, 607317 (3)"<br><span class="commentTitle">OMIM_ID :</span> 608877<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:12290086-12572098"><div class="tooltip">1:12290086-12572098<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:12290086-12572098<span class="commentTitle">P_loss_coord :</span> 1:12290086-12572098<br><span class="commentTitle">P_loss_source :</span> morbid:VPS13D<br><span class="commentTitle">P_loss_phen :</span> "Spinocerebellar ataxia, autosomal recessive 4, 607317 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:12290086-12572098"><div class="tooltip">1:12290086-12572098<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:12290086-12572098<span class="commentTitle">P_loss_coord :</span> 1:12290086-12572098</span></div>	</td>
	<td data-search="3"><div class="tooltip">3<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 3<span class="commentTitle">P_snvindel_nb :</span> 3<br><span class="commentTitle">P_snvindel_phen :</span> Autosomal_recessive_cerebellar_ataxia-saccadic_intrusion_syndrome</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="121" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF3300" data-search="TNFRSF8"><div class="tooltip">TNFRSF8<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TNFRSF8<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> "8,64E+03"<br><span class="commentTitle">ExAC_pLI :</span> "8,46E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 88.92<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -1.479.625.895<br><span class="commentTitle">ExAC_delZ :</span> 0.743807760285324<br><span class="commentTitle">ExAC_dupZ :</span> 1.795.101.223<br><span class="commentTitle">ExAC_synZ :</span> 0.138161196139221<br><span class="commentTitle">ExAC_misZ :</span> 0.574247822963843<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001243<br><span class="commentTitle">Exon_count :</span> 15<br><span class="commentTitle">Overlapped_tx_length :</span> 80905<br><span class="commentTitle">Overlapped_CDS_length :</span> 1788<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="122" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="PARK7"><div class="tooltip">PARK7<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PARK7<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "7,53E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,04E+03"<br><span class="commentTitle">HI :</span> 30<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 10.87<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.792105787757176<br><span class="commentTitle">ExAC_delZ :</span> 0.623219057166178<br><span class="commentTitle">ExAC_dupZ :</span> 1.848.222.016<br><span class="commentTitle">ExAC_synZ :</span> -0.169572204968715<br><span class="commentTitle">ExAC_misZ :</span> -0.427014314862717<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_007262<br><span class="commentTitle">Exon_count :</span> 7<br><span class="commentTitle">Overlapped_tx_length :</span> 23795<br><span class="commentTitle">Overlapped_CDS_length :</span> 570<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Parkinson disease 7, AR early-onset, 606324 (3); Parkinson disease 7, AR early-onset, 606324 (3)""><div class="tooltip">"Parkinson disease 7, AR early-onset, 606324 [...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Parkinson disease 7, AR early-onset, 606324 (3); Parkinson disease 7, AR early-onset, 606324 (3)"<br><span class="commentTitle">OMIM_ID :</span> 602533; 602533<br><span class="commentTitle">OMIM_inheritance :</span> AR; AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:8021770-8045565"><div class="tooltip">1:8021770-8045565<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:8021770-8045565<span class="commentTitle">P_loss_coord :</span> 1:8021770-8045565<br><span class="commentTitle">P_loss_source :</span> morbid:PARK7<br><span class="commentTitle">P_loss_phen :</span> "Parkinson disease 7, autosomal recessive early-onset, 606324 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:8021770-8045565"><div class="tooltip">1:8021770-8045565<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:8021770-8045565<span class="commentTitle">P_loss_coord :</span> 1:8021770-8045565</span></div>	</td>
	<td data-search="3"><div class="tooltip">3<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 3<span class="commentTitle">P_snvindel_nb :</span> 3<br><span class="commentTitle">P_snvindel_phen :</span> Amyotrophic_lateral_sclerosis-parkinsonism-dementia_complex; Parkinson_disease_7</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="123" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="ICMT"><div class="tooltip">ICMT<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ICMT<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "7,94E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,59E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 35.62<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.986010913429058<br><span class="commentTitle">ExAC_delZ :</span> 0.522953872733126<br><span class="commentTitle">ExAC_dupZ :</span> 0.853665508878739<br><span class="commentTitle">ExAC_synZ :</span> 0.249918380163028<br><span class="commentTitle">ExAC_misZ :</span> -1.710.766.550<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_012405<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 14772<br><span class="commentTitle">Overlapped_CDS_length :</span> 855<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="124" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="TNFRSF1B"><div class="tooltip">TNFRSF1B<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TNFRSF1B<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "5,02E+03"<br><span class="commentTitle">ExAC_pLI :</span> "7,79E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 79<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.805271555284089<br><span class="commentTitle">ExAC_delZ :</span> 0.653218446480238<br><span class="commentTitle">ExAC_dupZ :</span> 0.614568682536779<br><span class="commentTitle">ExAC_synZ :</span> 0.316966599161352<br><span class="commentTitle">ExAC_misZ :</span> 2.079.505.170<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001066<br><span class="commentTitle">Exon_count :</span> 10<br><span class="commentTitle">Overlapped_tx_length :</span> 42230<br><span class="commentTitle">Overlapped_CDS_length :</span> 1386<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="125" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="NOL9"><div class="tooltip">NOL9<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NOL9<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "8,88E+01"<br><span class="commentTitle">ExAC_pLI :</span> "4,47E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 84.4<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.316400802950708<br><span class="commentTitle">ExAC_delZ :</span> -0.421233645900279<br><span class="commentTitle">ExAC_dupZ :</span> -0.272696609256104<br><span class="commentTitle">ExAC_synZ :</span> -0.14502643144282<br><span class="commentTitle">ExAC_misZ :</span> -1.632.315.603<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_024654<br><span class="commentTitle">Exon_count :</span> 12<br><span class="commentTitle">Overlapped_tx_length :</span> 33167<br><span class="commentTitle">Overlapped_CDS_length :</span> 2109<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="126" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="PLEKHG5"><div class="tooltip">PLEKHG5<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PLEKHG5<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "3,36E-02"<br><span class="commentTitle">ExAC_pLI :</span> "4,03E-01"<br><span class="commentTitle">HI :</span> 30<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 53.41<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.0305710111342706<br><span class="commentTitle">ExAC_delZ :</span> -0.0131643134990412<br><span class="commentTitle">ExAC_dupZ :</span> 0.0409539789372569<br><span class="commentTitle">ExAC_synZ :</span> -0.568126884031604<br><span class="commentTitle">ExAC_misZ :</span> 0.628000952499697<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001265593<br><span class="commentTitle">Exon_count :</span> 21<br><span class="commentTitle">Overlapped_tx_length :</span> 19378<br><span class="commentTitle">Overlapped_CDS_length :</span> 3228<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Spinal muscular atrophy, distal, AR, 4, 611067 (3)/  Charcot-Marie-Tooth disease, recessive intermediate C, 615376 (3)""><div class="tooltip">"Spinal muscular atrophy, distal, AR, 4, 6110[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Spinal muscular atrophy, distal, AR, 4, 611067 (3)/  Charcot-Marie-Tooth disease, recessive intermediate C, 615376 (3)"<br><span class="commentTitle">OMIM_ID :</span> 611101<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:6534120-6534211"><div class="tooltip">1:6534120-6534211<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:6534120-6534211<span class="commentTitle">P_loss_coord :</span> 1:6534120-6534211<br><span class="commentTitle">P_loss_source :</span> CLN:628239<br><span class="commentTitle">P_loss_phen :</span> "Distal_spinal_muscular_atrophy,_autosomal_recessive_4|Charcot-Marie-Tooth_disease,_recessive_intermediate_c"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:6534120-6534211"><div class="tooltip">1:6534120-6534211<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:6534120-6534211<span class="commentTitle">P_loss_coord :</span> 1:6534120-6534211</span></div>	</td>
	<td data-search="10"><div class="tooltip">10<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 10<span class="commentTitle">P_snvindel_nb :</span> 10<br><span class="commentTitle">P_snvindel_phen :</span> "Charcot-Marie-Tooth_disease,_recessive_intermediate_c; Distal_spinal_muscular_atrophy,_autosomal_recessive_4; Juvenile_amyotrophic_lateral_sclerosis"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="127" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="SRM"><div class="tooltip">SRM<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SRM<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "2,35E+03"<br><span class="commentTitle">ExAC_pLI :</span> "8,70E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 42.31<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.452645703671854<br><span class="commentTitle">ExAC_delZ :</span> 0.288503706159159<br><span class="commentTitle">ExAC_dupZ :</span> 0.339840203142772<br><span class="commentTitle">ExAC_synZ :</span> 0.512748136542023<br><span class="commentTitle">ExAC_misZ :</span> 1.864.552.012<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_003132<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 5430<br><span class="commentTitle">Overlapped_CDS_length :</span> 909<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="128" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="ACOT7"><div class="tooltip">ACOT7<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ACOT7<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "7,59E+03"<br><span class="commentTitle">ExAC_pLI :</span> "3,82E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 36.89<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -1.128.335.140<br><span class="commentTitle">ExAC_delZ :</span> -632.908.205<br><span class="commentTitle">ExAC_dupZ :</span> 0.56492762227724<br><span class="commentTitle">ExAC_synZ :</span> 0.0670724558623954<br><span class="commentTitle">ExAC_misZ :</span> 1.613.845.923<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_181864<br><span class="commentTitle">Exon_count :</span> 9<br><span class="commentTitle">Overlapped_tx_length :</span> 121497<br><span class="commentTitle">Overlapped_CDS_length :</span> 1143<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="129" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="PEX14"><div class="tooltip">PEX14<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PEX14<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> "2,26E+03"<br><span class="commentTitle">ExAC_pLI :</span> "6,35E+03"<br><span class="commentTitle">HI :</span> 30<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 19.48<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.0707395393971832<br><span class="commentTitle">ExAC_delZ :</span> 0.126966559082228<br><span class="commentTitle">ExAC_dupZ :</span> -0.00304834174343369<br><span class="commentTitle">ExAC_synZ :</span> 0.605545467807364<br><span class="commentTitle">ExAC_misZ :</span> -750.520.821<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> PEROXISOME BIOGENESIS DISORDER COMPLEMENTATION GROUP K<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_004565<br><span class="commentTitle">Exon_count :</span> 9<br><span class="commentTitle">Overlapped_tx_length :</span> 155809<br><span class="commentTitle">Overlapped_CDS_length :</span> 1134<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Peroxisome biogenesis disorder 13A (Zellweger), 614887 (3)""><div class="tooltip">"Peroxisome biogenesis disorder 13A (Zellwege[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Peroxisome biogenesis disorder 13A (Zellweger), 614887 (3)"<br><span class="commentTitle">OMIM_ID :</span> 601791<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:10535006-10690815"><div class="tooltip">1:10535006-10690815<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:10535006-10690815<span class="commentTitle">P_loss_coord :</span> 1:10535006-10690815<br><span class="commentTitle">P_loss_source :</span> morbid:PEX14<br><span class="commentTitle">P_loss_phen :</span> "Peroxisome biogenesis disorder 13A (Zellweger), 614887 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:10535006-10690815"><div class="tooltip">1:10535006-10690815<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:10535006-10690815<span class="commentTitle">P_loss_coord :</span> 1:10535006-10690815</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="130" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="TNFRSF9"><div class="tooltip">TNFRSF9<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TNFRSF9<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "3,22E+03"<br><span class="commentTitle">ExAC_pLI :</span> "2,53E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 93.3<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.843669223715674<br><span class="commentTitle">ExAC_delZ :</span> 0.702458980914178<br><span class="commentTitle">ExAC_dupZ :</span> 0.619812162984905<br><span class="commentTitle">ExAC_synZ :</span> 0.0157748094432195<br><span class="commentTitle">ExAC_misZ :</span> -0.415420829092102<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001561<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 24969<br><span class="commentTitle">Overlapped_CDS_length :</span> 768<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="131" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="SLC45A1"><div class="tooltip">SLC45A1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SLC45A1<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "2,48E-01"<br><span class="commentTitle">ExAC_pLI :</span> "4,52E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 57.66<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.545387102129997<br><span class="commentTitle">ExAC_delZ :</span> 0.690923550631244<br><span class="commentTitle">ExAC_dupZ :</span> 0.289952978931426<br><span class="commentTitle">ExAC_synZ :</span> -1.790.227.675<br><span class="commentTitle">ExAC_misZ :</span> 178.938.390<br><span class="commentTitle">DDD_status :</span> probable<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> all missense/ in frame<br><span class="commentTitle">DDD_disease :</span> Intellectual disability and epilepsy<br><span class="commentTitle">DDD_pmid :</span> 28434495</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001379614<br><span class="commentTitle">Exon_count :</span> 9<br><span class="commentTitle">Overlapped_tx_length :</span> 26052<br><span class="commentTitle">Overlapped_CDS_length :</span> 2271<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Intellectual developmental disorder with neuropsychiatric features, 617532 (3)""><div class="tooltip">"Intellectual developmental disorder with neu[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Intellectual developmental disorder with neuropsychiatric features, 617532 (3)"<br><span class="commentTitle">OMIM_ID :</span> 605763<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:8378173-8404225"><div class="tooltip">1:8378173-8404225<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:8378173-8404225<span class="commentTitle">P_loss_coord :</span> 1:8378173-8404225<br><span class="commentTitle">P_loss_source :</span> morbid:SLC45A1<br><span class="commentTitle">P_loss_phen :</span> "Intellectual developmental disorder with neuropsychiatric features, 617532 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:8378173-8404225"><div class="tooltip">1:8378173-8404225<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:8378173-8404225<span class="commentTitle">P_loss_coord :</span> 1:8378173-8404225</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="132" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="ZBTB48"><div class="tooltip">ZBTB48<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ZBTB48<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "1,15E+00"<br><span class="commentTitle">ExAC_pLI :</span> "6,80E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 48.25<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.256591381099148<br><span class="commentTitle">ExAC_delZ :</span> 0.512768558716477<br><span class="commentTitle">ExAC_dupZ :</span> -0.590433566422884<br><span class="commentTitle">ExAC_synZ :</span> -0.841906085956462<br><span class="commentTitle">ExAC_misZ :</span> 1.438.118.877<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001278647<br><span class="commentTitle">Exon_count :</span> 11<br><span class="commentTitle">Overlapped_tx_length :</span> 9287<br><span class="commentTitle">Overlapped_CDS_length :</span> 2067<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="133" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="MAD2L2"><div class="tooltip">MAD2L2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MAD2L2<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "8,21E+02"<br><span class="commentTitle">ExAC_pLI :</span> "2,03E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 5.39<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 1.073.629.813<br><span class="commentTitle">ExAC_delZ :</span> 0.587309845215093<br><span class="commentTitle">ExAC_dupZ :</span> 0.92713652820267<br><span class="commentTitle">ExAC_synZ :</span> 321.466.920<br><span class="commentTitle">ExAC_misZ :</span> 433.445.816<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001127325<br><span class="commentTitle">Exon_count :</span> 9<br><span class="commentTitle">Overlapped_tx_length :</span> 17142<br><span class="commentTitle">Overlapped_CDS_length :</span> 636<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""?Fanconi anemia, complementation group V, 617243 (3)""><div class="tooltip">"?Fanconi anemia, complementation group V, 61[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "?Fanconi anemia, complementation group V, 617243 (3)"<br><span class="commentTitle">OMIM_ID :</span> 604094<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:11734536-11751678"><div class="tooltip">1:11734536-11751678<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:11734536-11751678<span class="commentTitle">P_loss_coord :</span> 1:11734536-11751678<br><span class="commentTitle">P_loss_source :</span> morbid:MAD2L2<br><span class="commentTitle">P_loss_phen :</span> "?Fanconi anemia, complementation group V, 617243 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:11734536-11751678"><div class="tooltip">1:11734536-11751678<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:11734536-11751678<span class="commentTitle">P_loss_coord :</span> 1:11734536-11751678</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="134" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="CLCN6"><div class="tooltip">CLCN6<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CLCN6<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "6,11E-04"<br><span class="commentTitle">ExAC_pLI :</span> "1,64E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 33.73<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.846216947981292<br><span class="commentTitle">ExAC_delZ :</span> 1.912.936.282<br><span class="commentTitle">ExAC_dupZ :</span> 0.463266607454407<br><span class="commentTitle">ExAC_synZ :</span> 0.0818637259708723<br><span class="commentTitle">ExAC_misZ :</span> 1.298.575.625<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001286<br><span class="commentTitle">Exon_count :</span> 23<br><span class="commentTitle">Overlapped_tx_length :</span> 36940<br><span class="commentTitle">Overlapped_CDS_length :</span> 2610<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="135" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="DHRS3"><div class="tooltip">DHRS3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> DHRS3<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "2,26E+03"<br><span class="commentTitle">ExAC_pLI :</span> "6,61E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 18.45<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -306.924.401<br><span class="commentTitle">ExAC_delZ :</span> 0.429642306150646<br><span class="commentTitle">ExAC_dupZ :</span> -1.341.704.709<br><span class="commentTitle">ExAC_synZ :</span> 0.789102613272088<br><span class="commentTitle">ExAC_misZ :</span> -1.909.508.899<br><span class="commentTitle">DDD_status :</span> possible<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> cis-regulatory or promotor mutation<br><span class="commentTitle">DDD_disease :</span> DHRS3 related craniosynostosis<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_004753<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 50278<br><span class="commentTitle">Overlapped_CDS_length :</span> 909<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="136" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="EXOSC10"><div class="tooltip">EXOSC10<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> EXOSC10<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "1,97E-10"<br><span class="commentTitle">ExAC_pLI :</span> "1,66E-03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 49.57<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.128083564606386<br><span class="commentTitle">ExAC_delZ :</span> -0.326780358987852<br><span class="commentTitle">ExAC_dupZ :</span> -0.0796263611820062<br><span class="commentTitle">ExAC_synZ :</span> 0.209244581752745<br><span class="commentTitle">ExAC_misZ :</span> 0.649834503560985<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001001998<br><span class="commentTitle">Exon_count :</span> 25<br><span class="commentTitle">Overlapped_tx_length :</span> 33251<br><span class="commentTitle">Overlapped_CDS_length :</span> 2658<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="137" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="SPSB1"><div class="tooltip">SPSB1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SPSB1<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "5,20E+03"<br><span class="commentTitle">ExAC_pLI :</span> "7,10E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 28.43<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.647101973222973<br><span class="commentTitle">ExAC_delZ :</span> 0.0220556668691239<br><span class="commentTitle">ExAC_dupZ :</span> 0.835517864431618<br><span class="commentTitle">ExAC_synZ :</span> -0.102584545129288<br><span class="commentTitle">ExAC_misZ :</span> -1.553.694.346<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_025106<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 76638<br><span class="commentTitle">Overlapped_CDS_length :</span> 822<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="138" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="TNFRSF25"><div class="tooltip">TNFRSF25<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TNFRSF25<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "8,65E+01"<br><span class="commentTitle">ExAC_pLI :</span> "4,49E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 71.65<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.305475865395656<br><span class="commentTitle">ExAC_delZ :</span> -0.11056019220228<br><span class="commentTitle">ExAC_dupZ :</span> 0.451624089682139<br><span class="commentTitle">ExAC_synZ :</span> -125.190.911<br><span class="commentTitle">ExAC_misZ :</span> -47.834.593<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_148965<br><span class="commentTitle">Exon_count :</span> 10<br><span class="commentTitle">Overlapped_tx_length :</span> 5042<br><span class="commentTitle">Overlapped_CDS_length :</span> 1281<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="139" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="RBP7"><div class="tooltip">RBP7<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RBP7<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> "5,41E+03"<br><span class="commentTitle">ExAC_pLI :</span> "4,09E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 66.12<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.639939484627037<br><span class="commentTitle">ExAC_delZ :</span> -0.696733969212168<br><span class="commentTitle">ExAC_dupZ :</span> -0.538496602487543<br><span class="commentTitle">ExAC_synZ :</span> -0.289677385217051<br><span class="commentTitle">ExAC_misZ :</span> 0.209378416431013<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_052960<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 18794<br><span class="commentTitle">Overlapped_CDS_length :</span> 405<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="140" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="FBXO6"><div class="tooltip">FBXO6<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> FBXO6<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> "1,52E+02"<br><span class="commentTitle">ExAC_pLI :</span> "4,86E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 75.95<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.95967597951439<br><span class="commentTitle">ExAC_delZ :</span> 0.493830664648047<br><span class="commentTitle">ExAC_dupZ :</span> 0.847170578976804<br><span class="commentTitle">ExAC_synZ :</span> 0.559284087469729<br><span class="commentTitle">ExAC_misZ :</span> -0.254032857482549<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_018438<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 10153<br><span class="commentTitle">Overlapped_CDS_length :</span> 882<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="141" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="PER3"><div class="tooltip">PER3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PER3<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> "1,37E-13"<br><span class="commentTitle">ExAC_pLI :</span> "2,19E-06"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 84.56<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.463403971735157<br><span class="commentTitle">ExAC_delZ :</span> -0.359137568071167<br><span class="commentTitle">ExAC_dupZ :</span> 0.929808348206803<br><span class="commentTitle">ExAC_synZ :</span> -0.503339232642588<br><span class="commentTitle">ExAC_misZ :</span> -0.89270060070442<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001377275<br><span class="commentTitle">Exon_count :</span> 22<br><span class="commentTitle">Overlapped_tx_length :</span> 60887<br><span class="commentTitle">Overlapped_CDS_length :</span> 3633<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""?Advanced sleep phase syndrome, familial, 3, 616882 (3)""><div class="tooltip">"?Advanced sleep phase syndrome, familial, 3,[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "?Advanced sleep phase syndrome, familial, 3, 616882 (3)"<br><span class="commentTitle">OMIM_ID :</span> 603427<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:7844488-7905237"><div class="tooltip">1:7844488-7905237<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:7844488-7905237<span class="commentTitle">P_loss_coord :</span> 1:7844488-7905237<br><span class="commentTitle">P_loss_source :</span> morbid:PER3<br><span class="commentTitle">P_loss_phen :</span> "?Advanced sleep phase syndrome, familial, 3, 616882 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:7844488-7905237"><div class="tooltip">1:7844488-7905237<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:7844488-7905237<span class="commentTitle">P_loss_coord :</span> 1:7844488-7905237</span></div>	</td>
	<td data-search="2"><div class="tooltip">2<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 2<span class="commentTitle">P_snvindel_nb :</span> 2<br><span class="commentTitle">P_snvindel_phen :</span> "Advanced_sleep_phase_syndrome,_familial,_3"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="142" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="ERRFI1"><div class="tooltip">ERRFI1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ERRFI1<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> "1,22E+02"<br><span class="commentTitle">ExAC_pLI :</span> "4,66E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 74.64<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.628417882512621<br><span class="commentTitle">ExAC_delZ :</span> 1.919.518.301<br><span class="commentTitle">ExAC_dupZ :</span> 0.149614770517327<br><span class="commentTitle">ExAC_synZ :</span> -0.375491506943176<br><span class="commentTitle">ExAC_misZ :</span> 0.252821709234483<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_018948<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 14583<br><span class="commentTitle">Overlapped_CDS_length :</span> 1389<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="143" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="NMNAT1"><div class="tooltip">NMNAT1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NMNAT1<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> "3,54E+02"<br><span class="commentTitle">ExAC_pLI :</span> "2,25E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 62.8<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.123056552974546<br><span class="commentTitle">ExAC_delZ :</span> -0.344636831029443<br><span class="commentTitle">ExAC_dupZ :</span> -0.019532106261306<br><span class="commentTitle">ExAC_synZ :</span> -0.599663839058627<br><span class="commentTitle">ExAC_misZ :</span> -0.721418745276475<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> LEBER CONGENITAL AMAUROSIS<br><span class="commentTitle">DDD_pmid :</span> 22842227; 12734549; 22842230</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001297778<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 42576<br><span class="commentTitle">Overlapped_CDS_length :</span> 840<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Leber congenital amaurosis 9, 608553 (3)""><div class="tooltip">"Leber congenital amaurosis 9, 608553 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Leber congenital amaurosis 9, 608553 (3)"<br><span class="commentTitle">OMIM_ID :</span> 608700<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:10003532-10045556"><div class="tooltip">1:10003532-10045556<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:10003532-10045556<span class="commentTitle">P_loss_coord :</span> 1:10003532-10045556<br><span class="commentTitle">P_loss_source :</span> morbid:NMNAT1<br><span class="commentTitle">P_loss_phen :</span> "Leber congenital amaurosis 9, 608553 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:10003532-10045556"><div class="tooltip">1:10003532-10045556<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:10003532-10045556<span class="commentTitle">P_loss_coord :</span> 1:10003532-10045556</span></div>	</td>
	<td data-search="13"><div class="tooltip">13<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 13<span class="commentTitle">P_snvindel_nb :</span> 13<br><span class="commentTitle">P_snvindel_phen :</span> Cone-rod_dystrophy; Developmental_regression; Diarrhea; Gastrointestinal_dysmotility; Global_developmental_delay; Leber_congenital_amaurosis; Leber_congenital_amaurosis_9; Nystagmus; Retinal_dystrophy; Severe_visual_impairment</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="144" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="ENO1"><div class="tooltip">ENO1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ENO1<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> "1,91E-01"<br><span class="commentTitle">ExAC_pLI :</span> "1,04E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 1.52<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.570521374454392<br><span class="commentTitle">ExAC_delZ :</span> -0.398299491729596<br><span class="commentTitle">ExAC_dupZ :</span> -0.705423808364236<br><span class="commentTitle">ExAC_synZ :</span> 0.677399125863915<br><span class="commentTitle">ExAC_misZ :</span> 0.830216009266221<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001353346<br><span class="commentTitle">Exon_count :</span> 12<br><span class="commentTitle">Overlapped_tx_length :</span> 17687<br><span class="commentTitle">Overlapped_CDS_length :</span> 1305<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="Enolase deficiency (1)"><div class="tooltip">Enolase deficiency (1)<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> Enolase deficiency (1)<br><span class="commentTitle">OMIM_ID :</span> 172430<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:8921058-8938745"><div class="tooltip">1:8921058-8938745<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:8921058-8938745<span class="commentTitle">P_loss_coord :</span> 1:8921058-8938745<br><span class="commentTitle">P_loss_source :</span> morbid:ENO1<br><span class="commentTitle">P_loss_phen :</span> Enolase deficiency (1)<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:8921058-8938745"><div class="tooltip">1:8921058-8938745<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:8921058-8938745<span class="commentTitle">P_loss_coord :</span> 1:8921058-8938745</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="145" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="KIAA2013"><div class="tooltip">KIAA2013<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KIAA2013<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> "9,78E+01"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 61.81<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.0466317837582463<br><span class="commentTitle">ExAC_delZ :</span> -0.311789129046188<br><span class="commentTitle">ExAC_dupZ :</span> 0.199101769130702<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_138346<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 6837<br><span class="commentTitle">Overlapped_CDS_length :</span> 1905<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="146" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="SLC25A33"><div class="tooltip">SLC25A33<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SLC25A33<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> "5,92E+01"<br><span class="commentTitle">ExAC_pLI :</span> "4,51E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 38.98<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.0511899372742477<br><span class="commentTitle">ExAC_delZ :</span> 0.121757520495409<br><span class="commentTitle">ExAC_dupZ :</span> -0.0532234218457971<br><span class="commentTitle">ExAC_synZ :</span> -0.381566342574674<br><span class="commentTitle">ExAC_misZ :</span> 683.073.083<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_032315<br><span class="commentTitle">Exon_count :</span> 7<br><span class="commentTitle">Overlapped_tx_length :</span> 45708<br><span class="commentTitle">Overlapped_CDS_length :</span> 966<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="147" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="C1orf127"><div class="tooltip">C1orf127<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C1orf127<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,49E-07"<br><span class="commentTitle">ExAC_pLI :</span> "1,33E-01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.14<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.269635066124169<br><span class="commentTitle">ExAC_delZ :</span> 0.397275411362107<br><span class="commentTitle">ExAC_dupZ :</span> 0.0732059225662102<br><span class="commentTitle">ExAC_synZ :</span> -0.452474598581702<br><span class="commentTitle">ExAC_misZ :</span> -0.953286576790673<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001170754<br><span class="commentTitle">Exon_count :</span> 13<br><span class="commentTitle">Overlapped_tx_length :</span> 35565<br><span class="commentTitle">Overlapped_CDS_length :</span> 2472<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="148" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="H6PD"><div class="tooltip">H6PD<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> H6PD<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "8,59E-05"<br><span class="commentTitle">ExAC_pLI :</span> "1,11E+00"<br><span class="commentTitle">HI :</span> 30<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 46.01<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -306.924.401<br><span class="commentTitle">ExAC_delZ :</span> -0.129940873548319<br><span class="commentTitle">ExAC_dupZ :</span> -1.341.704.709<br><span class="commentTitle">ExAC_synZ :</span> 0.260307309300236<br><span class="commentTitle">ExAC_misZ :</span> -0.444048951064682<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001282587<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 31489<br><span class="commentTitle">Overlapped_CDS_length :</span> 2409<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Cortisone reductase deficiency 1, 604931 (3)""><div class="tooltip">"Cortisone reductase deficiency 1, 604931 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Cortisone reductase deficiency 1, 604931 (3)"<br><span class="commentTitle">OMIM_ID :</span> 138090<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:9299905-9331394"><div class="tooltip">1:9299905-9331394<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:9299905-9331394<span class="commentTitle">P_loss_coord :</span> 1:9299905-9331394<br><span class="commentTitle">P_loss_source :</span> morbid:H6PD<br><span class="commentTitle">P_loss_phen :</span> "Cortisone reductase deficiency 1, 604931 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:9299905-9331394"><div class="tooltip">1:9299905-9331394<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:9299905-9331394<span class="commentTitle">P_loss_coord :</span> 1:9299905-9331394</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="149" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="CA6"><div class="tooltip">CA6<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CA6<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "3,60E+00"<br><span class="commentTitle">ExAC_pLI :</span> "8,02E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 93.66<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.702378027775052<br><span class="commentTitle">ExAC_delZ :</span> -0.692544130405288<br><span class="commentTitle">ExAC_dupZ :</span> -0.663582294763541<br><span class="commentTitle">ExAC_synZ :</span> 0.157698257167856<br><span class="commentTitle">ExAC_misZ :</span> -0.160379180632044<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001270500<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 28577<br><span class="commentTitle">Overlapped_CDS_length :</span> 942<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="150" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="UBIAD1"><div class="tooltip">UBIAD1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> UBIAD1<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,95E+02"<br><span class="commentTitle">ExAC_pLI :</span> "5,69E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 37.45<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.824876131030435<br><span class="commentTitle">ExAC_delZ :</span> 0.378475529346678<br><span class="commentTitle">ExAC_dupZ :</span> 0.727158008968833<br><span class="commentTitle">ExAC_synZ :</span> -0.214250343577772<br><span class="commentTitle">ExAC_misZ :</span> 1.513.268.698<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_013319<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 15237<br><span class="commentTitle">Overlapped_CDS_length :</span> 1017<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Corneal dystrophy, Schnyder type, 121800 (3)""><div class="tooltip">"Corneal dystrophy, Schnyder type, 121800 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Corneal dystrophy, Schnyder type, 121800 (3)"<br><span class="commentTitle">OMIM_ID :</span> 611632<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="151" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="NPHP4"><div class="tooltip">NPHP4<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NPHP4<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,74E-25"<br><span class="commentTitle">ExAC_pLI :</span> "1,29E-13"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 68.34<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.531255410051606<br><span class="commentTitle">ExAC_delZ :</span> 0.458012077584659<br><span class="commentTitle">ExAC_dupZ :</span> -0.887803279570691<br><span class="commentTitle">ExAC_synZ :</span> -261.311.986<br><span class="commentTitle">ExAC_misZ :</span> 1.281.119.279<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> NEPHRONOPHTHISIS TYPE 4<br><span class="commentTitle">DDD_pmid :</span> 12244321</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_015102<br><span class="commentTitle">Exon_count :</span> 30<br><span class="commentTitle">Overlapped_tx_length :</span> 129615<br><span class="commentTitle">Overlapped_CDS_length :</span> 4281<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Nephronophthisis 4, 606966 (3)/  Senior-Loken syndrome 4, 606996 (3)""><div class="tooltip">"Nephronophthisis 4, 606966 (3)/  Senior-Loke[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Nephronophthisis 4, 606966 (3)/  Senior-Loken syndrome 4, 606996 (3)"<br><span class="commentTitle">OMIM_ID :</span> 607215<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:5922870-6052485"><div class="tooltip">1:5922870-6052485<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:5922870-6052485<span class="commentTitle">P_loss_coord :</span> 1:5922870-6052485<br><span class="commentTitle">P_loss_source :</span> morbid:NPHP4<br><span class="commentTitle">P_loss_phen :</span> "Nephronophthisis 4, 606966 (3); Senior-Loken syndrome 4, 606996 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:5922870-6052485"><div class="tooltip">1:5922870-6052485<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:5922870-6052485<span class="commentTitle">P_loss_coord :</span> 1:5922870-6052485</span></div>	</td>
	<td data-search="19"><div class="tooltip">19<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 19<span class="commentTitle">P_snvindel_nb :</span> 19<br><span class="commentTitle">P_snvindel_phen :</span> Chronic_kidney_disease; Hematuria; Nephronophthisis; Nephronophthisis_4; Proteinuria; Senior-Loken_syndrome_4</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="152" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="PLOD1"><div class="tooltip">PLOD1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PLOD1<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,48E-11"<br><span class="commentTitle">ExAC_pLI :</span> "2,70E-03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 29.26<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.493334038005503<br><span class="commentTitle">ExAC_delZ :</span> 1.278.038.676<br><span class="commentTitle">ExAC_dupZ :</span> 1.712.641.251<br><span class="commentTitle">ExAC_synZ :</span> 0.118398436840632<br><span class="commentTitle">ExAC_misZ :</span> 0.399681556777185<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> "EHLERS-DANLOS SYNDROME, KYPHOSCOLIOTIC FORM"<br><span class="commentTitle">DDD_pmid :</span> 8163671; 5016372; 1345174; 9450904</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001316320<br><span class="commentTitle">Exon_count :</span> 20<br><span class="commentTitle">Overlapped_tx_length :</span> 40821<br><span class="commentTitle">Overlapped_CDS_length :</span> 2325<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Ehlers-Danlos syndrome, kyphoscoliotic type, 1, 225400 (3)""><div class="tooltip">"Ehlers-Danlos syndrome, kyphoscoliotic type,[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Ehlers-Danlos syndrome, kyphoscoliotic type, 1, 225400 (3)"<br><span class="commentTitle">OMIM_ID :</span> 153454<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:11994773-12035594"><div class="tooltip">1:11994773-12035594<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:11994773-12035594<span class="commentTitle">P_loss_coord :</span> 1:11994773-12035594<br><span class="commentTitle">P_loss_source :</span> morbid:PLOD1<br><span class="commentTitle">P_loss_phen :</span> "Ehlers-Danlos syndrome, kyphoscoliotic type, 1, 225400 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:11994773-12035594"><div class="tooltip">1:11994773-12035594<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:11994773-12035594<span class="commentTitle">P_loss_coord :</span> 1:11994773-12035594</span></div>	</td>
	<td data-search="19"><div class="tooltip">19<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 19<span class="commentTitle">P_snvindel_nb :</span> 19<br><span class="commentTitle">P_snvindel_phen :</span> "Acquired_porencephaly; Bilateral_cryptorchidism; Cardiovascular_phenotype; Congenital_omphalocele; Depressed_nasal_bridge; Dolichocephaly; Ehlers-Danlos_syndrome,_hydroxylysine-deficient; Feeding_difficulties; Generalized_hypotonia; Generalized_neonatal_hypotonia; High_palate; Hydrocephalus; Joint_hypermobility; Macrocephaly_at_birth; Narrow_chest; Neonatal_hypotonia; Scrotal_hypoplasia; Severe_global_developmental_delay; Short_chin; Thoracolumbar_scoliosis; Umbilical_hernia"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="153" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="MTHFR"><div class="tooltip">MTHFR<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MTHFR<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "3,21E-06"<br><span class="commentTitle">ExAC_pLI :</span> "2,46E-03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 7.44<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.591874266196786<br><span class="commentTitle">ExAC_delZ :</span> -0.0248062312728537<br><span class="commentTitle">ExAC_dupZ :</span> 0.854953981725245<br><span class="commentTitle">ExAC_synZ :</span> -0.676475975674101<br><span class="commentTitle">ExAC_misZ :</span> 366.338.568<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> biallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> METHYLENETETRAHYDROFOLATE REDUCTASE DEFICIENCY<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001330358<br><span class="commentTitle">Exon_count :</span> 12<br><span class="commentTitle">Overlapped_tx_length :</span> 17948<br><span class="commentTitle">Overlapped_CDS_length :</span> 2094<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""(Vascular disease, susceptibility to) (3)/  (Schizophrenia, susceptibility to), 181500 (3)/  Homocystinuria due to MTHFR deficiency, 236250 (3)/  (Neural tube defects, susceptibility to), 601634 (3)/  (Thromboembolism, susceptibility to), 188050 (3)""><div class="tooltip">"(Vascular disease, susceptibility to) (3)/  [...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "(Vascular disease, susceptibility to) (3)/  (Schizophrenia, susceptibility to), 181500 (3)/  Homocystinuria due to MTHFR deficiency, 236250 (3)/  (Neural tube defects, susceptibility to), 601634 (3)/  (Thromboembolism, susceptibility to), 188050 (3)"<br><span class="commentTitle">OMIM_ID :</span> 607093<br><span class="commentTitle">OMIM_inheritance :</span> / AD/ AR/ AR/ AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:11845786-11863734"><div class="tooltip">1:11845786-11863734<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:11845786-11863734<span class="commentTitle">P_loss_coord :</span> 1:11845786-11863734<br><span class="commentTitle">P_loss_source :</span> morbid:MTHFR<br><span class="commentTitle">P_loss_phen :</span> "Homocystinuria due to MTHFR deficiency, 236250 (3); Neural tube defects, susceptibility to, 601634 (3); Schizophrenia, susceptibility to, 181500 (3); Thromboembolism, susceptibility to, 188050 (3); Vascular disease, susceptibility to (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:11845786-11863734"><div class="tooltip">1:11845786-11863734<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:11845786-11863734<span class="commentTitle">P_loss_coord :</span> 1:11845786-11863734</span></div>	</td>
	<td data-search="17"><div class="tooltip">17<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 17<span class="commentTitle">P_snvindel_nb :</span> 17<br><span class="commentTitle">P_snvindel_phen :</span> "Homocystinuria_due_to_MTHFR_deficiency; Homocystinuria_due_to_methylene_tetrahydrofolate_reductase_deficiency; Neural_tube_defects,_folate-sensitive"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="154" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="AGTRAP"><div class="tooltip">AGTRAP<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> AGTRAP<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,88E+03"<br><span class="commentTitle">ExAC_pLI :</span> "4,17E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 79.47<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.0516778754989619<br><span class="commentTitle">ExAC_delZ :</span> -0.669057823256601<br><span class="commentTitle">ExAC_dupZ :</span> 0.33629036631979<br><span class="commentTitle">ExAC_synZ :</span> -0.61110115599059<br><span class="commentTitle">ExAC_misZ :</span> 0.366126610675045<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_020350<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 14634<br><span class="commentTitle">Overlapped_CDS_length :</span> 480<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="155" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="RNF207"><div class="tooltip">RNF207<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RNF207<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "5,47E-08"<br><span class="commentTitle">ExAC_pLI :</span> "5,19E-02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 62.23<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.194137475909732<br><span class="commentTitle">ExAC_delZ :</span> -0.457285899882981<br><span class="commentTitle">ExAC_dupZ :</span> 0.560254141530725<br><span class="commentTitle">ExAC_synZ :</span> 1.956.969.935<br><span class="commentTitle">ExAC_misZ :</span> -0.0984671193959529<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_207396<br><span class="commentTitle">Exon_count :</span> 18<br><span class="commentTitle">Overlapped_tx_length :</span> 15181<br><span class="commentTitle">Overlapped_CDS_length :</span> 1905<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="156" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="RPL22"><div class="tooltip">RPL22<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RPL22<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "1,56E+03"<br><span class="commentTitle">ExAC_pLI :</span> "3,48E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 31.35<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.71853421212648<br><span class="commentTitle">ExAC_delZ :</span> 0.227465276807663<br><span class="commentTitle">ExAC_dupZ :</span> 0.663891428792773<br><span class="commentTitle">ExAC_synZ :</span> 0.759972527283371<br><span class="commentTitle">ExAC_misZ :</span> -297.881.622<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_000983<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 14576<br><span class="commentTitle">Overlapped_CDS_length :</span> 387<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="157" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="FBXO2"><div class="tooltip">FBXO2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> FBXO2<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "2,15E+00"<br><span class="commentTitle">ExAC_pLI :</span> "2,73E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 63.55<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.958027616417935<br><span class="commentTitle">ExAC_delZ :</span> 0.502996047354977<br><span class="commentTitle">ExAC_dupZ :</span> 0.852771805865658<br><span class="commentTitle">ExAC_synZ :</span> 2.086.909.510<br><span class="commentTitle">ExAC_misZ :</span> 851.939.457<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_012168<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 6043<br><span class="commentTitle">Overlapped_CDS_length :</span> 891<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="158" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="C1orf167"><div class="tooltip">C1orf167<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C1orf167<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "9,15E-30"<br><span class="commentTitle">ExAC_pLI :</span> "1,56E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.01<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 0.4005535890877<br><span class="commentTitle">ExAC_misZ :</span> 0.505944350749215<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001010881<br><span class="commentTitle">Exon_count :</span> 21<br><span class="commentTitle">Overlapped_tx_length :</span> 27393<br><span class="commentTitle">Overlapped_CDS_length :</span> 4350<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="159" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFF00" data-search="ESPN"><div class="tooltip">ESPN<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ESPN<br><span class="commentTitle">LOEUF_bin :</span> 5<br><span class="commentTitle">GnomAD_pLI :</span> "2,87E-05"<br><span class="commentTitle">ExAC_pLI :</span> "2,55E+01"<br><span class="commentTitle">HI :</span> 30<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 46.09<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.332816766372613<br><span class="commentTitle">ExAC_delZ :</span> -0.0816551003020419<br><span class="commentTitle">ExAC_dupZ :</span> 0.468276536099848<br><span class="commentTitle">ExAC_synZ :</span> -113.755.411<br><span class="commentTitle">ExAC_misZ :</span> 0.0552908413911689<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_031475<br><span class="commentTitle">Exon_count :</span> 13<br><span class="commentTitle">Overlapped_tx_length :</span> 36169<br><span class="commentTitle">Overlapped_CDS_length :</span> 2565<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""?Usher syndrome, type 1M, 618632 (3)/  Deafness, AR 36, 609006 (3)/  Deafness, neurosensory, without vestibular involvement, AD, 609006 (3)""><div class="tooltip">"?Usher syndrome, type 1M, 618632 (3)/  Deafn[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "?Usher syndrome, type 1M, 618632 (3)/  Deafness, AR 36, 609006 (3)/  Deafness, neurosensory, without vestibular involvement, AD, 609006 (3)"<br><span class="commentTitle">OMIM_ID :</span> 606351<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:6484835-6521004"><div class="tooltip">1:6484835-6521004<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:6484835-6521004<span class="commentTitle">P_loss_coord :</span> 1:6484835-6521004<br><span class="commentTitle">P_loss_source :</span> morbid:ESPN<br><span class="commentTitle">P_loss_phen :</span> "?Usher syndrome, type 1M, 618632 (3); Deafness, autosomal recessive 36, 609006 (3); Deafness, neurosensory, without vestibular involvement, autosomal dominant, 609006 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:6484835-6521004"><div class="tooltip">1:6484835-6521004<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:6484835-6521004<span class="commentTitle">P_loss_coord :</span> 1:6484835-6521004</span></div>	</td>
	<td data-search="3"><div class="tooltip">3<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 3<span class="commentTitle">P_snvindel_nb :</span> 3<br><span class="commentTitle">P_snvindel_phen :</span> "Deafness,_autosomal_recessive_36,_with_or_without_vestibular_involvement; USHER_SYNDROME,_TYPE_1M; Usher_syndrome_type_1"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="160" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="LZIC"><div class="tooltip">LZIC<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LZIC<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "1,17E+01"<br><span class="commentTitle">ExAC_pLI :</span> "1,98E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 20.35<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.464271979614765<br><span class="commentTitle">ExAC_delZ :</span> 0.511680493146838<br><span class="commentTitle">ExAC_dupZ :</span> 0.318806053947926<br><span class="commentTitle">ExAC_synZ :</span> 0.480366861888472<br><span class="commentTitle">ExAC_misZ :</span> 0.347775973765208<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001316974<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 17099<br><span class="commentTitle">Overlapped_CDS_length :</span> 636<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="161" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="HES3"><div class="tooltip">HES3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HES3<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "2,77E+03"<br><span class="commentTitle">ExAC_pLI :</span> "2,69E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 78.15<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.671238595948328<br><span class="commentTitle">ExAC_delZ :</span> 0.178058205881183<br><span class="commentTitle">ExAC_dupZ :</span> 0.621272724105392<br><span class="commentTitle">ExAC_synZ :</span> 2.070.737.412<br><span class="commentTitle">ExAC_misZ :</span> -1.264.980.628<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001024598<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 1400<br><span class="commentTitle">Overlapped_CDS_length :</span> 561<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="162" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="GPR157"><div class="tooltip">GPR157<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> GPR157<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "9,75E+00"<br><span class="commentTitle">ExAC_pLI :</span> "2,63E-01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 67.42<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.0862096681686439<br><span class="commentTitle">ExAC_delZ :</span> -0.481305809605088<br><span class="commentTitle">ExAC_dupZ :</span> 0.1297920705571<br><span class="commentTitle">ExAC_synZ :</span> 973.637.502<br><span class="commentTitle">ExAC_misZ :</span> 0.282785595777459<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_024980<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 28798<br><span class="commentTitle">Overlapped_CDS_length :</span> 1008<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="163" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="MASP2"><div class="tooltip">MASP2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MASP2<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "3,04E-10"<br><span class="commentTitle">ExAC_pLI :</span> "1,58E-10"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 72.87<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -306.924.401<br><span class="commentTitle">ExAC_delZ :</span> -632.908.205<br><span class="commentTitle">ExAC_dupZ :</span> -2.125.411.349<br><span class="commentTitle">ExAC_synZ :</span> 1.015.223.592<br><span class="commentTitle">ExAC_misZ :</span> 254.040.777<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_006610<br><span class="commentTitle">Exon_count :</span> 11<br><span class="commentTitle">Overlapped_tx_length :</span> 20717<br><span class="commentTitle">Overlapped_CDS_length :</span> 2061<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""MASP2 deficiency, 613791 (3)""><div class="tooltip">"MASP2 deficiency, 613791 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "MASP2 deficiency, 613791 (3)"<br><span class="commentTitle">OMIM_ID :</span> 605102<br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:11104854-11107296"><div class="tooltip">1:11104854-11107296<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:11104854-11107296<span class="commentTitle">P_loss_coord :</span> 1:11104854-11107296<br><span class="commentTitle">P_loss_source :</span> morbid:MASP2<br><span class="commentTitle">P_loss_phen :</span> "MASP2 deficiency, 613791 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:11104854-11107296"><div class="tooltip">1:11104854-11107296<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:11104854-11107296<span class="commentTitle">P_loss_coord :</span> 1:11104854-11107296</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="164" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="DFFA"><div class="tooltip">DFFA<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> DFFA<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "8,66E-01"<br><span class="commentTitle">ExAC_pLI :</span> "2,18E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 63.88<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.577123633983987<br><span class="commentTitle">ExAC_delZ :</span> 755.789.023<br><span class="commentTitle">ExAC_dupZ :</span> 0.303889182264447<br><span class="commentTitle">ExAC_synZ :</span> 1.861.123.925<br><span class="commentTitle">ExAC_misZ :</span> 0.630983916674337<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_004401<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 16008<br><span class="commentTitle">Overlapped_CDS_length :</span> 996<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="165" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="TAS1R1"><div class="tooltip">TAS1R1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TAS1R1<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "7,34E-10"<br><span class="commentTitle">ExAC_pLI :</span> "7,91E-05"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 76.37<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.360381121128725<br><span class="commentTitle">ExAC_delZ :</span> 0.433304614370224<br><span class="commentTitle">ExAC_dupZ :</span> -0.69245657334377<br><span class="commentTitle">ExAC_synZ :</span> -0.096283952883136<br><span class="commentTitle">ExAC_misZ :</span> 0.374112103171369<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_138697<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 24449<br><span class="commentTitle">Overlapped_CDS_length :</span> 2526<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="166" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="FBXO44"><div class="tooltip">FBXO44<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> FBXO44<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "2,11E-02"<br><span class="commentTitle">ExAC_pLI :</span> "3,60E-01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 38.55<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.530476096615816<br><span class="commentTitle">ExAC_delZ :</span> -0.326456595504181<br><span class="commentTitle">ExAC_dupZ :</span> 0.977025330025151<br><span class="commentTitle">ExAC_synZ :</span> -1.337.063.127<br><span class="commentTitle">ExAC_misZ :</span> -1.251.249.150<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001330355<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 8471<br><span class="commentTitle">Overlapped_CDS_length :</span> 771<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="167" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="SLC2A5"><div class="tooltip">SLC2A5<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SLC2A5<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "3,57E-06"<br><span class="commentTitle">ExAC_pLI :</span> "5,68E-01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 75.26<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 246.101.221<br><span class="commentTitle">ExAC_delZ :</span> 2.068.478.911<br><span class="commentTitle">ExAC_dupZ :</span> 1.362.592.013<br><span class="commentTitle">ExAC_synZ :</span> 1.953.637.401<br><span class="commentTitle">ExAC_misZ :</span> 2.134.778.293<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001328619<br><span class="commentTitle">Exon_count :</span> 13<br><span class="commentTitle">Overlapped_tx_length :</span> 37154<br><span class="commentTitle">Overlapped_CDS_length :</span> 1506<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="168" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="PDPN"><div class="tooltip">PDPN<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PDPN<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "3,20E+00"<br><span class="commentTitle">ExAC_pLI :</span> "9,84E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 83.14<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.287315948079754<br><span class="commentTitle">ExAC_delZ :</span> 0.115975608195528<br><span class="commentTitle">ExAC_dupZ :</span> 0.278176850960584<br><span class="commentTitle">ExAC_synZ :</span> -0.376651003010527<br><span class="commentTitle">ExAC_misZ :</span> -0.54178965077317<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_198389<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 34201<br><span class="commentTitle">Overlapped_CDS_length :</span> 711<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="169" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="LRRC38"><div class="tooltip">LRRC38<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LRRC38<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "1,44E+03"<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 51.16<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001010847<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 39058<br><span class="commentTitle">Overlapped_CDS_length :</span> 885<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="170" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="KLHL21"><div class="tooltip">KLHL21<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KLHL21<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> "6,31E-03"<br><span class="commentTitle">ExAC_pLI :</span> "1,08E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 45.34<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.492316201128964<br><span class="commentTitle">ExAC_delZ :</span> 0.0879201994484844<br><span class="commentTitle">ExAC_dupZ :</span> -0.702734830788034<br><span class="commentTitle">ExAC_synZ :</span> 0.628522427253427<br><span class="commentTitle">ExAC_misZ :</span> -743.253.342<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_014851<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 12146<br><span class="commentTitle">Overlapped_CDS_length :</span> 1794<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="171" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="GPR153"><div class="tooltip">GPR153<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> GPR153<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "6,88E-03"<br><span class="commentTitle">ExAC_pLI :</span> "3,78E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 57.44<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.464804447487487<br><span class="commentTitle">ExAC_delZ :</span> -0.204582747392109<br><span class="commentTitle">ExAC_dupZ :</span> 0.708418671117821<br><span class="commentTitle">ExAC_synZ :</span> 0.768442627357824<br><span class="commentTitle">ExAC_misZ :</span> 0.946182041463517<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_207370<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 13746<br><span class="commentTitle">Overlapped_CDS_length :</span> 1830<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="172" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="THAP3"><div class="tooltip">THAP3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> THAP3<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "7,28E+01"<br><span class="commentTitle">ExAC_pLI :</span> "1,48E+01"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 78.57<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.297136328105604<br><span class="commentTitle">ExAC_delZ :</span> -0.268039750128885<br><span class="commentTitle">ExAC_dupZ :</span> -0.285426511747504<br><span class="commentTitle">ExAC_synZ :</span> -0.421338828397795<br><span class="commentTitle">ExAC_misZ :</span> 0.762247041069098<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001195753<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 8718<br><span class="commentTitle">Overlapped_CDS_length :</span> 720<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="173" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="CTNNBIP1"><div class="tooltip">CTNNBIP1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CTNNBIP1<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "2,48E+02"<br><span class="commentTitle">ExAC_pLI :</span> "2,30E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 22.5<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.636565157191764<br><span class="commentTitle">ExAC_delZ :</span> -0.0441111929629715<br><span class="commentTitle">ExAC_dupZ :</span> 0.846282717838518<br><span class="commentTitle">ExAC_synZ :</span> 0.267816167204162<br><span class="commentTitle">ExAC_misZ :</span> 0.618580982906945<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001012329<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 61994<br><span class="commentTitle">Overlapped_CDS_length :</span> 246<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="174" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="PRAMEF12"><div class="tooltip">PRAMEF12<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF12<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "5,27E-01"<br><span class="commentTitle">ExAC_pLI :</span> "3,90E-02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 99.34<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 741.987.255<br><span class="commentTitle">ExAC_misZ :</span> -613.027.278<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001080830<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 4168<br><span class="commentTitle">Overlapped_CDS_length :</span> 1452<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="175" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="VAMP3"><div class="tooltip">VAMP3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> VAMP3<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "2,59E+01"<br><span class="commentTitle">ExAC_pLI :</span> "5,43E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 32.37<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.440165189908552<br><span class="commentTitle">ExAC_delZ :</span> 0.298358153808152<br><span class="commentTitle">ExAC_dupZ :</span> 0.373310124253111<br><span class="commentTitle">ExAC_synZ :</span> -0.392205486009828<br><span class="commentTitle">ExAC_misZ :</span> 0.541782372518335<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_004781<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 10137<br><span class="commentTitle">Overlapped_CDS_length :</span> 303<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="176" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="UTS2"><div class="tooltip">UTS2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> UTS2<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "3,16E+02"<br><span class="commentTitle">ExAC_pLI :</span> "1,51E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 80.56<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.181682462935049<br><span class="commentTitle">ExAC_delZ :</span> -0.605789903730052<br><span class="commentTitle">ExAC_dupZ :</span> 0.651817607732903<br><span class="commentTitle">ExAC_synZ :</span> 0.750618139574096<br><span class="commentTitle">ExAC_misZ :</span> -0.115640050575395<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_021995<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 5880<br><span class="commentTitle">Overlapped_CDS_length :</span> 420<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="177" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="PRAMEF11"><div class="tooltip">PRAMEF11<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF11<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> "6,16E+00"<br><span class="commentTitle">ExAC_pLI :</span> "2,85E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 98.32<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 686.251.821<br><span class="commentTitle">ExAC_misZ :</span> 1.297.484.757<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001146344<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 6802<br><span class="commentTitle">Overlapped_CDS_length :</span> 1436<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> yes<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1083"><div class="tooltip">1000g; DDD:446; DDD:450; DDD:455; DDD:456; dg[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1083<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1083<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12848806-12903525; 1:12853865-12919353; 1:12854390-13146372; 1:12854895-12919194; 1:12854895-12919194; 1:12856327-12893193; 1:12856327-12893193; 1:12860749-12913896; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772</span></div>	</td>
	<td data-search="1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1083"><div class="tooltip">1000g; DDD:446; DDD:450; DDD:455; DDD:456; dg[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1083<span class="commentTitle">B_loss_source :</span> 1000g; DDD:446; DDD:450; DDD:455; DDD:456; dgv6e214; dgv8n27; esv2421662; esv3585241; esv3585242; gnomAD-SV_v2.1_DEL_1_1083<br><span class="commentTitle">B_loss_coord :</span> 1:12836474-12921834; 1:12841928-12971833; 1:12848806-12903525; 1:12853865-12919353; 1:12854390-13146372; 1:12854895-12919194; 1:12854895-12919194; 1:12856327-12893193; 1:12856327-12893193; 1:12860749-12913896; 1:12877849-12933342; 1:12877893-12933272; 1:12877893-12948772</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="178" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="MIIP"><div class="tooltip">MIIP<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIIP<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "1,31E-16"<br><span class="commentTitle">ExAC_pLI :</span> "1,96E-11"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 88.63<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.264906693478186<br><span class="commentTitle">ExAC_delZ :</span> 0.505831535598632<br><span class="commentTitle">ExAC_dupZ :</span> -0.584999808653335<br><span class="commentTitle">ExAC_synZ :</span> 0.0917492697232198<br><span class="commentTitle">ExAC_misZ :</span> -0.767746009372014<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_021933<br><span class="commentTitle">Exon_count :</span> 10<br><span class="commentTitle">Overlapped_tx_length :</span> 12548<br><span class="commentTitle">Overlapped_CDS_length :</span> 1167<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="179" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="NPPA"><div class="tooltip">NPPA<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NPPA<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "5,45E+00"<br><span class="commentTitle">ExAC_pLI :</span> "1,69E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 5.02<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 2.369.557<br><span class="commentTitle">ExAC_delZ :</span> 1.148.814.473<br><span class="commentTitle">ExAC_dupZ :</span> 1.834.539.659<br><span class="commentTitle">ExAC_synZ :</span> -0.582708653110477<br><span class="commentTitle">ExAC_misZ :</span> 337.271.474<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_006172<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 2075<br><span class="commentTitle">Overlapped_CDS_length :</span> 456<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Atrial standstill 2, 615745 (3)/  Atrial fibrillation, familial, 6, 612201 (3)""><div class="tooltip">"Atrial standstill 2, 615745 (3)/  Atrial fib[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Atrial standstill 2, 615745 (3)/  Atrial fibrillation, familial, 6, 612201 (3)"<br><span class="commentTitle">OMIM_ID :</span> 108780<br><span class="commentTitle">OMIM_inheritance :</span> AR/ AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:11905765-11907840"><div class="tooltip">1:11905765-11907840<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:11905765-11907840<span class="commentTitle">P_loss_coord :</span> 1:11905765-11907840<br><span class="commentTitle">P_loss_source :</span> morbid:NPPA<br><span class="commentTitle">P_loss_phen :</span> "Atrial fibrillation, familial, 6, 612201 (3); Atrial standstill 2, 615745 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:11905765-11907840"><div class="tooltip">1:11905765-11907840<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:11905765-11907840<span class="commentTitle">P_loss_coord :</span> 1:11905765-11907840</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="180" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="HES2"><div class="tooltip">HES2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HES2<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "2,63E+01"<br><span class="commentTitle">ExAC_pLI :</span> "6,70E+02"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 77.46<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 2.002.454.780<br><span class="commentTitle">ExAC_misZ :</span> 1.752.786.908<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_019089<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 4686<br><span class="commentTitle">Overlapped_CDS_length :</span> 522<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="181" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="PRAMEF10"><div class="tooltip">PRAMEF10<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRAMEF10<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "1,11E+01"<br><span class="commentTitle">ExAC_pLI :</span> "8,76E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 99.05<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -630.454.975<br><span class="commentTitle">ExAC_misZ :</span> -101.885.235<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001039361<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5375<br><span class="commentTitle">Overlapped_CDS_length :</span> 1425<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700; 1:13384749-13419500</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128"><div class="tooltip">gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL[...]<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<span class="commentTitle">B_loss_source :</span> gnomAD-SV_v2.1_DEL_1_1116; gnomAD-SV_v2.1_DEL_1_1126; gnomAD-SV_v2.1_DEL_1_1128<br><span class="commentTitle">B_loss_coord :</span> 1:13212290-13779616; 1:13379799-13497700; 1:13384749-13419500</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="182" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="ANGPTL7"><div class="tooltip">ANGPTL7<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ANGPTL7<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "9,17E-13"<br><span class="commentTitle">ExAC_pLI :</span> "8,44E-05"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 40.97<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.975201389323906<br><span class="commentTitle">ExAC_delZ :</span> 0.513834758973858<br><span class="commentTitle">ExAC_dupZ :</span> 0.852456837030538<br><span class="commentTitle">ExAC_synZ :</span> -0.458584023914172<br><span class="commentTitle">ExAC_misZ :</span> -0.296397939907582<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_021146<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 6627<br><span class="commentTitle">Overlapped_CDS_length :</span> 1041<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="183" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="AADACL3"><div class="tooltip">AADACL3<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> AADACL3<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "1,19E+02"<br><span class="commentTitle">ExAC_pLI :</span> "1,56E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 92.94<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -306.924.401<br><span class="commentTitle">ExAC_delZ :</span> -0.0099947629380129<br><span class="commentTitle">ExAC_dupZ :</span> -1.341.704.709<br><span class="commentTitle">ExAC_synZ :</span> 0.0361612141503088<br><span class="commentTitle">ExAC_misZ :</span> -0.109500896056784<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001103170<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 12615<br><span class="commentTitle">Overlapped_CDS_length :</span> 1224<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="184" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="C1orf158"><div class="tooltip">C1orf158<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C1orf158<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "2,53E+00"<br><span class="commentTitle">ExAC_pLI :</span> "1,57E-04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 82.65<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -0.290306780277456<br><span class="commentTitle">ExAC_misZ :</span> -0.560276665968168<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_152290<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 17700<br><span class="commentTitle">Overlapped_CDS_length :</span> 585<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="185" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="DRAXIN"><div class="tooltip">DRAXIN<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> DRAXIN<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "1,89E-06"<br><span class="commentTitle">ExAC_pLI :</span> "6,64E-06"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 70.15<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.0549674231870791<br><span class="commentTitle">ExAC_delZ :</span> -0.697242505498958<br><span class="commentTitle">ExAC_dupZ :</span> 0.342985615630726<br><span class="commentTitle">ExAC_synZ :</span> 0.44310633950293<br><span class="commentTitle">ExAC_misZ :</span> 0.40148401997718<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_198545<br><span class="commentTitle">Exon_count :</span> 7<br><span class="commentTitle">Overlapped_tx_length :</span> 34148<br><span class="commentTitle">Overlapped_CDS_length :</span> 1050<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="186" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="AADACL4"><div class="tooltip">AADACL4<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> AADACL4<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "1,54E+00"<br><span class="commentTitle">ExAC_pLI :</span> "5,48E+00"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 96.7<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -306.924.401<br><span class="commentTitle">ExAC_delZ :</span> 458.643.230<br><span class="commentTitle">ExAC_dupZ :</span> -1.341.704.709<br><span class="commentTitle">ExAC_synZ :</span> 0.627786154935008<br><span class="commentTitle">ExAC_misZ :</span> 658.655.549<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001013630<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 22984<br><span class="commentTitle">Overlapped_CDS_length :</span> 1224<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="187" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="SLC2A7"><div class="tooltip">SLC2A7<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SLC2A7<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> "5,54E-17"<br><span class="commentTitle">ExAC_pLI :</span> "3,00E-13"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 82.87<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -620.482.934<br><span class="commentTitle">ExAC_delZ :</span> -635.540.645<br><span class="commentTitle">ExAC_dupZ :</span> -0.949038682578216<br><span class="commentTitle">ExAC_synZ :</span> 651.197.574<br><span class="commentTitle">ExAC_misZ :</span> -0.608132442357145<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_207420<br><span class="commentTitle">Exon_count :</span> 12<br><span class="commentTitle">Overlapped_tx_length :</span> 23046<br><span class="commentTitle">Overlapped_CDS_length :</span> 1539<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="188" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="MTOR"><div class="tooltip">MTOR<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MTOR<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 0.59<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -701.182.627<br><span class="commentTitle">ExAC_delZ :</span> -31.232.603<br><span class="commentTitle">ExAC_dupZ :</span> -226.182.596<br><span class="commentTitle">ExAC_synZ :</span> 0.302361492731988<br><span class="commentTitle">ExAC_misZ :</span> -2.071.842.270<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> monoallelic<br><span class="commentTitle">DDD_consequence :</span> all missense/ in frame<br><span class="commentTitle">DDD_disease :</span> Smith-Kingsmore syndrome<br><span class="commentTitle">DDD_pmid :</span> 28892148; 23934111</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_004958<br><span class="commentTitle">Exon_count :</span> 58<br><span class="commentTitle">Overlapped_tx_length :</span> 156017<br><span class="commentTitle">Overlapped_CDS_length :</span> 7650<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Smith-Kingsmore syndrome, 616638 (3)/  Focal cortical dysplasia, type II, somatic, 607341 (3)""><div class="tooltip">"Smith-Kingsmore syndrome, 616638 (3)/  Focal[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Smith-Kingsmore syndrome, 616638 (3)/  Focal cortical dysplasia, type II, somatic, 607341 (3)"<br><span class="commentTitle">OMIM_ID :</span> 601231<br><span class="commentTitle">OMIM_inheritance :</span> AD/ <br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:11166591-11322608"><div class="tooltip">1:11166591-11322608<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:11166591-11322608<span class="commentTitle">P_loss_coord :</span> 1:11166591-11322608<br><span class="commentTitle">P_loss_source :</span> morbid:MTOR<br><span class="commentTitle">P_loss_phen :</span> "Focal cortical dysplasia, type II, somatic, 607341 (3); Smith-Kingsmore syndrome, 616638 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:11166591-11322608"><div class="tooltip">1:11166591-11322608<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:11166591-11322608<span class="commentTitle">P_loss_coord :</span> 1:11166591-11322608</span></div>	</td>
	<td data-search="10"><div class="tooltip">10<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 10<span class="commentTitle">P_snvindel_nb :</span> 10<br><span class="commentTitle">P_snvindel_phen :</span> "Breast_neoplasm; CEBALID_syndrome; Focal_cortical_dysplasia_type_II; Glioblastoma; Inborn_genetic_diseases; Intellectual_disability,_severe; MTOR-related_megalencephaly_and_pigmentary_mosaicism_in_skin; Rare_genetic_intellectual_disability; Renal_cell_carcinoma,_papillary,_1; Smith-Kingsmore_syndrome; Transitional_cell_carcinoma_of_the_bladder"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="189" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="AJAP1"><div class="tooltip">AJAP1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> AJAP1<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "9,95E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,78E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 55.25<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.506571148517976<br><span class="commentTitle">ExAC_delZ :</span> 0.662552527971597<br><span class="commentTitle">ExAC_dupZ :</span> 0.243124085964785<br><span class="commentTitle">ExAC_synZ :</span> -1.680.436.767<br><span class="commentTitle">ExAC_misZ :</span> -401.161.006<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_018836<br><span class="commentTitle">Exon_count :</span> 6<br><span class="commentTitle">Overlapped_tx_length :</span> 137914<br><span class="commentTitle">Overlapped_CDS_length :</span> 1236<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="190" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="PHF13"><div class="tooltip">PHF13<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PHF13<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "9,80E+03"<br><span class="commentTitle">ExAC_pLI :</span> "9,31E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 43.25<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.0593301211268695<br><span class="commentTitle">ExAC_delZ :</span> 0.302095682559979<br><span class="commentTitle">ExAC_dupZ :</span> -0.284409948072345<br><span class="commentTitle">ExAC_synZ :</span> 168.075.353<br><span class="commentTitle">ExAC_misZ :</span> -2.089.992.876<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_153812<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 10300<br><span class="commentTitle">Overlapped_CDS_length :</span> 903<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="191" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="RERE"><div class="tooltip">RERE<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> RERE<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 1.92<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.363760929092707<br><span class="commentTitle">ExAC_delZ :</span> 0.643752189929251<br><span class="commentTitle">ExAC_dupZ :</span> 0.132941687190521<br><span class="commentTitle">ExAC_synZ :</span> 507.260.333<br><span class="commentTitle">ExAC_misZ :</span> 186.119.485<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> monoallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> Phenocopy of Proximal 1p36 Deletions<br><span class="commentTitle">DDD_pmid :</span> 27087320</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001042681<br><span class="commentTitle">Exon_count :</span> 23<br><span class="commentTitle">Overlapped_tx_length :</span> 465236<br><span class="commentTitle">Overlapped_CDS_length :</span> 4701<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Neurodevelopmental disorder with or without anomalies of the brain, eye, or heart, 616975 (3)""><div class="tooltip">"Neurodevelopmental disorder with or without [...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Neurodevelopmental disorder with or without anomalies of the brain, eye, or heart, 616975 (3)"<br><span class="commentTitle">OMIM_ID :</span> 605226<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:8412463-8877699"><div class="tooltip">1:8412463-8877699<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:8412463-8877699<span class="commentTitle">P_loss_coord :</span> 1:8412463-8877699<br><span class="commentTitle">P_loss_source :</span> morbid:RERE<br><span class="commentTitle">P_loss_phen :</span> "Neurodevelopmental disorder with or without anomalies of the brain, eye, or heart, 616975 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:8412463-8877699"><div class="tooltip">1:8412463-8877699<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:8412463-8877699<span class="commentTitle">P_loss_coord :</span> 1:8412463-8877699</span></div>	</td>
	<td data-search="10"><div class="tooltip">10<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 10<span class="commentTitle">P_snvindel_nb :</span> 10<br><span class="commentTitle">P_snvindel_phen :</span> "CHARGE_association; Inborn_genetic_diseases; Neurodevelopmental_disorder_with_or_without_anomalies_of_the_brain,_eye,_or_heart"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="192" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="PIK3CD"><div class="tooltip">PIK3CD<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PIK3CD<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 1.53<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.14144716232322<br><span class="commentTitle">ExAC_delZ :</span> 0.142333173010138<br><span class="commentTitle">ExAC_dupZ :</span> -0.296707676546266<br><span class="commentTitle">ExAC_synZ :</span> 0.156587129593956<br><span class="commentTitle">ExAC_misZ :</span> -407.751.147<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_005026<br><span class="commentTitle">Exon_count :</span> 24<br><span class="commentTitle">Overlapped_tx_length :</span> 77384<br><span class="commentTitle">Overlapped_CDS_length :</span> 3135<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Immunodeficiency 14, 615513 (3)""><div class="tooltip">"Immunodeficiency 14, 615513 (3)"<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Immunodeficiency 14, 615513 (3)"<br><span class="commentTitle">OMIM_ID :</span> 602839<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:9751524-9788980"><div class="tooltip">1:9751524-9788980<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:9751524-9788980<span class="commentTitle">P_loss_coord :</span> 1:9751524-9788980<br><span class="commentTitle">P_loss_source :</span> morbid:PIK3CD<br><span class="commentTitle">P_loss_phen :</span> "Immunodeficiency 14, 615513 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:9751524-9788980"><div class="tooltip">1:9751524-9788980<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:9751524-9788980<span class="commentTitle">P_loss_coord :</span> 1:9751524-9788980</span></div>	</td>
	<td data-search="2"><div class="tooltip">2<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 2<span class="commentTitle">P_snvindel_nb :</span> 2<br><span class="commentTitle">P_snvindel_phen :</span> Immunodeficiency_14; Inherited_Immunodeficiency_Diseases; not_specified</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="193" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="UBE4B"><div class="tooltip">UBE4B<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> UBE4B<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 25.55<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.42398976957681<br><span class="commentTitle">ExAC_delZ :</span> 0.402806222384478<br><span class="commentTitle">ExAC_dupZ :</span> 0.339668443248217<br><span class="commentTitle">ExAC_synZ :</span> 0.93482320007078<br><span class="commentTitle">ExAC_misZ :</span> 1.506.156.238<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001105562<br><span class="commentTitle">Exon_count :</span> 28<br><span class="commentTitle">Overlapped_tx_length :</span> 148281<br><span class="commentTitle">Overlapped_CDS_length :</span> 3909<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="194" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="PRDM2"><div class="tooltip">PRDM2<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PRDM2<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "9,95E+03"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 40.49<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.547664002453622<br><span class="commentTitle">ExAC_delZ :</span> 0.415153458557427<br><span class="commentTitle">ExAC_dupZ :</span> 0.373434796722033<br><span class="commentTitle">ExAC_synZ :</span> 1.272.818.822<br><span class="commentTitle">ExAC_misZ :</span> 0.436072998472952<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_012231<br><span class="commentTitle">Exon_count :</span> 10<br><span class="commentTitle">Overlapped_tx_length :</span> 120225<br><span class="commentTitle">Overlapped_CDS_length :</span> 5157<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="195" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="DNAJC11"><div class="tooltip">DNAJC11<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> DNAJC11<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "9,99E+03"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 33.21<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.267918910903994<br><span class="commentTitle">ExAC_delZ :</span> 0.0561661632510267<br><span class="commentTitle">ExAC_dupZ :</span> 0.318114344741748<br><span class="commentTitle">ExAC_synZ :</span> -0.490153239310217<br><span class="commentTitle">ExAC_misZ :</span> 1.333.729.937<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_018198<br><span class="commentTitle">Exon_count :</span> 16<br><span class="commentTitle">Overlapped_tx_length :</span> 67647<br><span class="commentTitle">Overlapped_CDS_length :</span> 1680<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="196" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="CAMTA1"><div class="tooltip">CAMTA1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CAMTA1<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> 3<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 2.22<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -1.694.898.607<br><span class="commentTitle">ExAC_delZ :</span> 0.837784888692585<br><span class="commentTitle">ExAC_dupZ :</span> -1.857.674.772<br><span class="commentTitle">ExAC_synZ :</span> 1.128.280.761<br><span class="commentTitle">ExAC_misZ :</span> -1.255.780.450<br><span class="commentTitle">DDD_status :</span> probable<br><span class="commentTitle">DDD_mode :</span> monoallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> "CEREBELLAR ATAXIA, NONPROGRESSIVE, WITH MENTAL RETARDATION"<br><span class="commentTitle">DDD_pmid :</span> 22693284</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_015215<br><span class="commentTitle">Exon_count :</span> 23<br><span class="commentTitle">Overlapped_tx_length :</span> 984253<br><span class="commentTitle">Overlapped_CDS_length :</span> 5022<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Cerebellar ataxia, nonprogressive, with mental retardation, 614756 (3)""><div class="tooltip">"Cerebellar ataxia, nonprogressive, with ment[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Cerebellar ataxia, nonprogressive, with mental retardation, 614756 (3)"<br><span class="commentTitle">OMIM_ID :</span> 611501<br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:7682000-7685000; 1:7764795-7829766"><div class="tooltip">1:7682000-7685000; 1:7764795-7829766<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:7682000-7685000; 1:7764795-7829766<span class="commentTitle">P_loss_coord :</span> 1:7682000-7685000; 1:7764795-7829766<br><span class="commentTitle">P_loss_source :</span> CLN:590571; morbid:CAMTA1<br><span class="commentTitle">P_loss_phen :</span> "Cerebellar ataxia, nonprogressive, with mental retardation, 614756 (3); Cerebellar_ataxia,_nonprogressive,_with_mental_retardation"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:7682000-7685000; 1:7764795-7829766"><div class="tooltip">1:7682000-7685000; 1:7764795-7829766<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:7682000-7685000; 1:7764795-7829766<span class="commentTitle">P_loss_coord :</span> 1:7682000-7685000; 1:7764795-7829766</span></div>	</td>
	<td data-search="16"><div class="tooltip">16<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 16<span class="commentTitle">P_snvindel_nb :</span> 16<br><span class="commentTitle">P_snvindel_phen :</span> "Cerebellar_ataxia,_nonprogressive,_with_mental_retardation; Inborn_genetic_diseases; Intellectual_disability"</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="197" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="CASZ1"><div class="tooltip">CASZ1<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CASZ1<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> 1<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 12.36<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.832497464985444<br><span class="commentTitle">ExAC_delZ :</span> 0.674871874874181<br><span class="commentTitle">ExAC_dupZ :</span> 0.646838432864827<br><span class="commentTitle">ExAC_synZ :</span> -1.852.791.740<br><span class="commentTitle">ExAC_misZ :</span> 338.729.700<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001079843<br><span class="commentTitle">Exon_count :</span> 21<br><span class="commentTitle">Overlapped_tx_length :</span> 160038<br><span class="commentTitle">Overlapped_CDS_length :</span> 5280<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="198" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="KIF1B"><div class="tooltip">KIF1B<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KIF1B<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 8.35<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.760981793752373<br><span class="commentTitle">ExAC_delZ :</span> 0.902664963441066<br><span class="commentTitle">ExAC_dupZ :</span> 0.565883591898392<br><span class="commentTitle">ExAC_synZ :</span> 0.587015354084205<br><span class="commentTitle">ExAC_misZ :</span> 1.070.325.553<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001365951<br><span class="commentTitle">Exon_count :</span> 49<br><span class="commentTitle">Overlapped_tx_length :</span> 171034<br><span class="commentTitle">Overlapped_CDS_length :</span> 5451<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search=""Pheochromocytoma, 171300 (3)/  ?Charcot-Marie-Tooth disease, type 2A1, 118210 (3)/  (Neuroblastoma, susceptibility to, 1), 256700 (3)""><div class="tooltip">"Pheochromocytoma, 171300 (3)/  ?Charcot-Mari[...]<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> "Pheochromocytoma, 171300 (3)/  ?Charcot-Marie-Tooth disease, type 2A1, 118210 (3)/  (Neuroblastoma, susceptibility to, 1), 256700 (3)"<br><span class="commentTitle">OMIM_ID :</span> 605995<br><span class="commentTitle">OMIM_inheritance :</span> "AD/ AD/ Somatic mutation, AD"<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="1:10290821-10368655"><div class="tooltip">1:10290821-10368655<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> 1:10290821-10368655<span class="commentTitle">P_loss_coord :</span> 1:10290821-10368655<br><span class="commentTitle">P_loss_source :</span> morbid:KIF1B<br><span class="commentTitle">P_loss_phen :</span> "?Charcot-Marie-Tooth disease, type 2A1, 118210 (3); Neuroblastoma, susceptibility to, 1, 256700 (3); Pheochromocytoma, 171300 (3)"<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="1:10290821-10368655"><div class="tooltip">1:10290821-10368655<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 1:10290821-10368655<span class="commentTitle">P_loss_coord :</span> 1:10290821-10368655</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
<tr class="fullsplit 1_4629201_15296111_DEL_1" >
	<td data-order="199" data-search="1_4629201_15296111_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr1:4629201-15296111&hgt.out1=submit&highlight=hg19.chr1:4629201-15296111#aaedff" target="_blank" rel="noopener noreferrer"">1_4629201_15296111_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :</span> 1_4629201_15296111_DEL_1<br><span class="commentTitle">SV_length :</span> -10666910</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="CHD5"><div class="tooltip">CHD5<span style="left: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> CHD5<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> "1,00E+04"<br><span class="commentTitle">ExAC_pLI :</span> "1,00E+04"<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 47.85<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.815573082077697<br><span class="commentTitle">ExAC_delZ :</span> 0.852507328630683<br><span class="commentTitle">ExAC_dupZ :</span> 0.659943539229556<br><span class="commentTitle">ExAC_synZ :</span> 0.245261442268261<br><span class="commentTitle">ExAC_misZ :</span> -1.978.581.101<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_015557<br><span class="commentTitle">Exon_count :</span> 42<br><span class="commentTitle">Overlapped_tx_length :</span> 78535<br><span class="commentTitle">Overlapped_CDS_length :</span> 5865<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_nearest_SS :</span> Absent in file<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_start :</span> Absent in file<br><span class="commentTitle">Intersect_end :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidate :</span> Absent in file</span></div>	</td>
	<td data-search="">.	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 12%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 25%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 37%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 50%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 62%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<span class="commentTitle">P_loss_coord :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 75%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 87%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> .</span></div>	</td>
</tr>
	</tbody>
	</table>
		</div>



	</body>
</html>